<!DOCTYPE html>
<!--[if IE 9]> <html lang="en" class="ie9"> <![endif]-->
<!--[if !IE]><!-->
<html lang="en">
<!--<![endif]-->
	<head>
		<meta name="google-site-verification" content="bzK5FySMBHqsDWYaHw6KeeAYY11QrOz0gUyP8iPx3Uw" />
		<meta charset="utf-8">
		<title>Tariq Faquih's Academic Webpage</title>
		<meta name="description" content="My personal webpage">
		<meta name="Tariq Faquih" content="">

		<!-- Mobile Meta -->
		<meta name="viewport" content="width=device-width, initial-scale=1.0">

		<!-- Favicon -->
		<link rel="shortcut icon" href="images/favicon.ico">
 
		<link href="./bootstrap/css/bootstrap.css" rel="stylesheet" type="text/css">  
		<link href="./fonts/font-awesome/css/font-awesome.css" rel="stylesheet" type="text/css"> 
		<link href="./fonts/academicons-1.9.1/css/academicons.css" rel="stylesheet" type="text/css"> 
		<link href="./css/animations.css" rel="stylesheet" type="text/css"> 
		<link href="./css/style.css" rel="stylesheet" type="text/css"> 
		<link href="./css/custom.css" rel="stylesheet" type="text/css">
		<!-- <link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro:300,400,700,300italic,400italic,700italic" rel="stylesheet"> -->

	</head>

	<body class="no-trans">
		<!-- scrollToTop --> 
		<div class="scrollToTop"><i class="icon-up-open-big"></i></div>

		<!-- header start --> 
		<header class="header fixed clearfix navbar navbar-fixed-top">
			<div class="container">
				<div class="row">
					<div class="col-md-4">

						<!-- header-left start --> 
						<div class="header-left">

							<!-- logo -->
							<div class="logo smooth-scroll">
								<a href="#banner"><img id="logo" src="images/logo2.png" alt="Worthy"></a>	
							</div>

							<!-- name-and-slogan -->
							<div class="logo-section smooth-scroll">
								<div class="brand"><a href="#banner">Tariq Faquih</a></div>								
							</div>

						</div>
						<!-- header-left end -->

					</div>
					<div class="col-md-8">

						<!-- header-right start --> 
						<div class="header-right">

							<!-- main-navigation start --> 
							<div class="main-navigation animated">

								<!-- navbar start --> 
								<nav class="navbar navbar-default" role="navigation">
									<div class="container-fluid">

										<!-- Toggle get grouped for better mobile display -->
										<div class="navbar-header">
											<button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#navbar-collapse-1">
												<span class="sr-only">Toggle navigation</span>
												<span class="icon-bar"></span>
												<span class="icon-bar"></span>
												<span class="icon-bar"></span>
											</button>
										</div>

										<!-- Collect the nav links, forms, and other content for toggling -->
										<div class="collapse navbar-collapse scrollspy smooth-scroll" id="navbar-collapse-1">
											<ul class="nav navbar-nav navbar-right">
												<li class="active"><a href="#banner">Home</a></li>
												<li><a href="#about">CV</a></li>
												<li><a href="#portfolio">Portfolio</a></li>
												<li><a href="Dissertation.html">Dissertation</a></li>
												

											</ul>
										</div>

									</div>
								</nav>
								<!-- navbar end -->

							</div>
							<!-- main-navigation end -->

						</div>
						<!-- header-right end -->

					</div>
				</div>
			</div>
		</header>
		<!-- header end -->

		<!-- banner start --> 
		<div id="banner" class="banner">
			<div class="banner-image"></div>
			<div class="banner-caption">
				<div class="container">
					<div class="row">					 
						<div class="caption-data" style="margin-top: 0px; opacity: 1;" data-animation-effect="fadeIn">
								<h1>Hello!<br>My Name is Tariq Faquih</h1>
								<p class="lead text-center">I am an Epidemiologist and Bioinformatician specialized in Genomics and Metabolomics. 
									<br>Welcome to my Academic Webpage.</p>
								<div class="padding-top60 contact-form">
								
									<ul class="social-links"  >
										<li class="github"><a target="_blank" href="https://github.com/tofaquih"><i class="fa fa-github"></i></a></li>
										<li class="twitter"><a target="_blank" href="https://twitter.com/TariqFaquih"><i class="fa fa-twitter"></i></a></li>
										<li class="google-scholar"><a target="_blank" href="https://scholar.google.nl/citations?hl=en&user=wYInss0AAAAJ"><i class="ai ai-google-scholar"></i></a></li>
										<li class="pubmed"><a target="_blank" href="https://pubmed.ncbi.nlm.nih.gov/?term=tariq+faquih&sort=date"><i class="ai ai-pubmed"></i></a></li>
										<li class="linkedin"><a target="_blank" href="https://www.linkedin.com/in/tariq-faquih-0b434782/"><i class="fa fa-linkedin"></i></a></li>
										<li class="orcid"><a target="_blank" href="https://orcid.org/0000-0001-8026-2251"><i class="ai ai-orcid"></i></a></li>
									</ul>
									
									

								</div>
							</div>
					</div>
				</div>
			</div>
		</div>
		<!-- banner end -->


		

 
		<!-- section start --> 
		<section class="section clearfix no-view secPadding" data-animation-effect="fadeIn">
			<div class="container">
				<div class="row">
					<div class="col-md-12">
						<h1 id="about" class="title text-center">About <span>Me</span></h1>
						<p class="lead text-center">I am a currently a Research Fellow at the Sleep Medicine Division at Brigham and Women's Hospital and I am finishing my PhD at the Clinical Epidemiology Department at Leiden University Medical Hospital.<br>
							<br>My focus is on epidemiological studies and methodologies using genomics, metabolomics, and proteomics.</p>
						<div class="space"></div>
						<div class="row">
							<div class="col-md-6">
								<img src="./images/MyPic.png" alt="">
								<div class="space"></div>
							</div>
							<div class="col-md-6">
								<p><b>My current research at the Sleep Medicine Division at Brigham and Women's Hospital and Harvard Medical School focuses on utilizing Omics for epidemiological studies on sleep disorders, such as Excessive Daytime Sleepiness (EDS), and health outcomes associated with sleep disorders.</b></p>
								<p>I received a scholarship for my PhD candidacy at the Department of Clinical Epidemiology, LUMC, Netherlands. <b>My PhD thesis, <u>"Multi-Omics in Research: Epidemiology, Methodology, and Advanced Data Analysis"</u></b>, focuses on using multiple Omics to study a variety of ailments. These Omics include genomics (Genome Wide Association Studies, Genomic Risk Scores), Metabolomics (large data from UHPLC-MS/MS and NMR platforms), and proteomics.</p>
								<p></p>
								<ul class="list-unstyled">
									<li><i class="fa fa-arrow-circle-right pr-10 colored"></i> Worked as a visiting researcher at the University of Cambridge (Department of Public Health and Primary care). It was a cooperative effort to perform an analysis for one of my ongoing projects on “metabolomics aging and disease prediction”.</li>
									<li><i class="fa fa-arrow-circle-right pr-10 colored"></i> Besides our direct projects we are also working on secondary projects with The Cohorts for Heart and Aging Research in Genomic Epidemiology (CHARGE) Consortium.</li>
									<li><i class="fa fa-arrow-circle-right pr-10 colored"></i> Organized and prioritized work to complete assignments in a timely, efficient manner.</li>
									<li><i class="fa fa-arrow-circle-right pr-10 colored"></i> Experienced in teaching courses for medical students. Courses include basic epidemiology, study designs, and mendelian randomization.</li>
								</ul>
							</div>
						</div>
						<div class="space"></div>
						<h2>Career and Education</h2>

						<div class="row">
							<div class="col-md-6">
									<ul class="list-unstyled">
										<p>From Saudi Arabia to England, to the United States, and to the Netherlands</p>
									<li><i class="fa fa-arrow-circle-right pr-10 colored"></i> 01/2023- Present : Research Fellow at Brigham Women's Hospital & Harvard Medical School</li>
									<li><i class="fa fa-arrow-circle-right pr-10 colored"></i> 08/2018- Present : PhD candidate at Leiden University Medical Center</li>
									<li><i class="fa fa-arrow-circle-right pr-10 colored"></i> 10/2013 - 08/2018: Bioinformatician • King Faisal Specialist Hospital and Research Center</li>
									<li><i class="fa fa-arrow-circle-right pr-10 colored"></i> 01/2012 - 06/2013: MSc Bioinformatics • 2013 • Georgetown University, Washington, D.C., USA</li>
									<li><i class="fa fa-arrow-circle-right pr-10 colored"></i> 09/2007 – 07/2011: BSc Human Genetics • 2011 • University of Leeds, Leeds, UK</li>

								</ul>
							</div>
							<div class="col-md-6">
								<div class="panel-group" id="accordion" role="tablist" aria-multiselectable="true">
									<div class="panel panel-default">
										<div class="panel-heading" role="tab" id="headingOne">
											<h4 class="panel-title">
												<a data-toggle="collapse" data-parent="#accordion" href="#collapseOne" aria-expanded="true" aria-controls="collapseOne">
													Personal Skills
												</a>
											</h4>
										</div>
										<div id="collapseOne" class="panel-collapse collapse in" role="tabpanel" aria-labelledby="headingOne">
											<div class="panel-body">
											<li>International experience and network</li>
											<li>Adaptability</li>
											<li>Fast Learner</li>
											<li>Critical thinking and problem solving</li>
											<li>Good teamwork, communication skills and ability to participate in research communities across organizational units.</li>
											</div>
										</div>
									</div>
									<div class="panel panel-default">
										<div class="panel-heading" role="tab" id="headingTwo">
											<h4 class="panel-title">
												<a class="collapsed" data-toggle="collapse" data-parent="#accordion" href="#collapseTwo" aria-expanded="false" aria-controls="collapseTwo">
													Epidemiology and Omics
												</a>
											</h4>
										</div>
										<div id="collapseTwo" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingTwo">
											<div class="panel-body">
												<li>Background in Epidemiology on cross sectional and longitudinal cohorts</li>
												<li>Background in computer science and bioinformatics</li>
												<li>Worked with genomic, proteomics, and metabolomics</li>
												<li>Experienced in Big Data Analysis</li>
												<li>Experienced in Missing Data Analysis</li>
												<li>Backgound in Genetics and working in the lab</li>
											</div>
										</div>
									</div>
									<div class="panel panel-default">
										<div class="panel-heading" role="tab" id="headingThree">
											<h4 class="panel-title">
												<a class="collapsed" data-toggle="collapse" data-parent="#accordion" href="#collapseThree" aria-expanded="false" aria-controls="collapseThree">
													Programming Experience
												</a>
											</h4>
										</div>
										<div id="collapseThree" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingThree">
											<div class="panel-body">
												<li>Proficient in Python and R programming languages</li>
												<li>Worked using Jupyter Notebook and R Studio</li>
												<li>Worked with STATA and SPSS</li>
												<li>Used several HPC clusters for big data analysis</li>
												<li>Worked with database resources such as SQL and Django</li>
												<li>Familiar with Git/GitHub, HTML5, Bash, Adobe Photoshop, Adobe Illustrator and Blender</li>
												<li>Experienced with Windows, MacOS and UNIX operating systems </li>

											</div>
										</div>
									</div>
								</div>
							</div>
						</div>


						
					</div>
				</div>
			</div>
		</section>
		<!-- section end -->

		<!-- section start --> 
		<div class="default-bg colord secPadding">
			<div class="container">

				</div>
			</div>
		</div>
		<!-- section end -->


		<!-- section start --> 
		<section class="section secPadding">
			<div class="container">
				<h1 class="text-center title" id="portfolio">Portfolio</h1>
				<div class="separator"></div>
				<p class="lead text-center">My projects revolve around the usage Multi-Omics to study a variety of phenotypes<br> 
				Here you can find projects on different topics that involved genomcis, proteomics, and metabolomics (and more).</p>
				<br>			
				<div class="row no-view" data-animation-effect="fadeIn">
					<div class="col-md-12">

						<!-- isotope filters start -->
						<div class="filters text-center">
							<ul class="nav nav-pills">
								<li class="active"><a href="#" data-filter="*">All</a></li>
								<li><a href="#" data-filter=".Genomics">Genomics</a></li>
								<li><a href="#" data-filter=".Metabolomics">Metabolomics</a></li>
								<li><a href="#" data-filter=".Proteomics">Proteomics</a></li>
								<li><a href="#" data-filter=".Methodology">Methodology</a></li>
								<li><a href="#" data-filter=".Etiology">Etiology</a></li>
								<li><a href="#" data-filter=".Prediction">Prediction</a></li>
							</ul>
						</div>
						<!-- isotope filters end -->




						



						<!-- portfolio items start -->
						<div class="isotope-container row grid-space-20">
							<div class="col-sm-6 col-md-3 isotope-item Metabolomics Methodology Genomics Proteomics Etiology Prediction">
								<div class="image-box">
									<div class="overlay-container">
										<img src="images/dissertation_thumb.png" alt="">
										<a class="overlay" data-toggle="modal" data-target="#project-1">
											<i class="fa fa-search-plus"></i>
											<a class="btn btn-default btn-block" data-toggle="modal" data-target="#project-1">Published</a>
										</a>
									</div>
									
									<h6 class="text-center">Multi-Omics in Research: Epidemiology, Methodology, and Advanced Data Analysis<br><br></h6> 
								</div>
								<!-- Modal -->
								<div class="modal fade" id="project-1" tabindex="-1" role="dialog" aria-labelledby="project-1-label" aria-hidden="true">
									<div class="modal-dialog modal-lg">
										<div class="modal-content">
											<div class="modal-header">
												<button type="button" class="close" data-dismiss="modal"><span aria-hidden="true">&times;</span><span class="sr-only">Close</span></button>
												<h4 class="modal-title" id="project-1-label">
													PhD Dissertation: Multi-Omics in Research: Epidemiology, Methodology, and Advanced Data Analysis (2023)</h4>
											</div>
											<div class="modal-body">
												
												<div class="row">												 
													<div class="col-md-12">
														<h7><i>PhD Dissertation: Multi-Omics in Research: Epidemiology, Methodology, and Advanced Data Analysis (2023)</i>
														</h7><br>
														<p><b>Summary</b><br>
															Metabolomics, proteomics, and genomics analyses provide profound insight into human biology
															and disease pathophysiology In this thesis, we explored the methodological challenges facing
															these OMICs technologies and illustrated their applications in epidemiological studies. In part
															one, we focused on some of the methodological challenges facing OMICs research. Chapter 2
															examined the agreement between aptamer-based proteomics measurements of venous thrombosis
															(VTE) biomarkers with those measured by standardized clinical instruments. We reported
															that 9 venous thrombosis biomarkers showed poor agreement with the clinical measurements.
															The agreement was particularly poor for D-dimer, a marker often used in VTE diagnosis. In
															Chapter 3, we explored the challenges of handling missing values in metabolomics data, particularly
															from untargeted metabolomics platforms. We assessed the performance of two previously
															reported methods for imputation, namely “k-nearest neighbor” (kNN) and “multiple imputation
															using chained equations”, by simulating missing patterns in data from the NEO study. Our
															findings showed that the multiple imputation method had less bias than the kNN method in
															most scenarios, except for the scenario with small sample size and high level of missingness.
															We further provided a publicly available R script to streamline the process of imputing missing
															metabolomics data using the two methods.
															Metabolites can provide unique insight into biological pathophysiology as they encompass the
															cumulative effects of genetic, lifestyle, and environmental factors. Thus, metabolomics analyses
															have been used to study biological processes such as aging. Some studies have used metabolomics
															to estimate “metabolomic age”, a reflection of aging on a metabolomic level, using
															prediction modelling methods. In Chapter 4, we addressed common challenges for developing
															metabolomics-based age prediction models and developed a model to predict metabolomic
															age using the Metabolon metabolomics platform measurements. We developed a model in the
															INTERVAL study using ridge regression and bootstrapping. This study population is a relatively
															healthy large population (n = 11,977) with a wide age range. Overall, after development and
															internal validation, our prediction models for metabolomic age demonstrated good performance
															(adjusted R2 = 0.83).
															The second part of the thesis addressed various epidemiological research questions by utilizing
															genomic data and metabolomics measurements (Metabolon and Nightingale platforms) and
															using advanced data analysis methods. In Chapter 5, we examined the associations of over 1300
															metabolites, measured by the Metabolon platform, with hepatic triglycerides content (HTGC).
															These associations included a range of endogenous metabolites from various pathways (particularly
															amino acids and lipids), as well as novel and uncharacterized metabolites. Subsequently,
															we constructed two complementary holistic networks, one biologically based and the other
															statistically driven, to assign the associated metabolites to pathways. These interactive networks
															revealed potential novel pathways associated with HTGC and possibly fatty liver diseases. We
															further provided these networks on an online, publicly available webpage for further exploration
															by the research community.
															One type of variation in the genome is variation in the number of repeated cytosine-adenine-
															guanine nucleotide sequences (CAG repeats). When the size of these repeats is beyond
															specific pathogenic thresholds in or near specific genes, they lead to the onset of several severe
															neurodegenerative diseases. One such case is the large CAG repeat sizes in the huntingtin (HTT)
															gene leading to the onset of Huntington’s disease. Interestingly, recent studies have found that
															CAG repeat sizes even within the non-pathogenic range are associated with health outcomes
															such as cognitive function, depression, and body weight. In Chapter 6, we explored the possible
															association between CAG repeat variation sizes below the pathogenic threshold (<36) in the
															HTT gene and the metabolite levels measured by the Nightingale platform. Multilevel mixed-effects
															and mediation analyses were conducted in pooled data from three large European cohorts
															(n = 10,275). Our results showed that large non-pathogenic CAG repeat sizes were associated
															with an unfavorable metabolomic profile, similar to that for an increased risk of cardiometabolic
															diseases.
															One of the advantages of metabolomics is the capability to measure “external” metabolites,
															often referred to as xenobiotics, from medication use, diet, and environmental exposures and
															contaminations. One such group of measured xenobiotics are the man-made per- and polyfluoroalkyl
															substances (PFAS)—commonly known as “forever chemicals” due to their persistence in
															the environment and the body. In addition to their persistent nature, high concentrations of PFAS
															have been reported to be associated with severe health impact. Despite efforts to regulate their
															production, PFAS levels remain detectable in the soil, drinking water, and human blood samples
															in the Netherlands and Germany. In Chapter 7, we studied the associations between common
															PFAS levels in blood, as measured by the Metabolon platform, and metabolomics and lipoprotein
															profiles (Nightingale platform) of the general population in the Netherlands (NEO study) and
															Germany (Rhineland study). Overall, PFAS metabolites were detectable in nearly all participants
															of both studies. Furthermore, as confirmed by meta-analysis, PFAS levels were associated with an
															unhealthy metabolomic profile characterized by increased fatty acids, low density lipoproteins,
															and apolipoprotein B levels. These associations are reminiscent of the metabolomic profile for
															increased risk of cardiometabolic diseases and were found to be particularly stronger in younger
															individuals. These data indicate that PFAS are a health risk even in individuals that apparently
															have not been exposed to high levels of PFAS.
															<br><br>
															
														</p>
													</div>
												</div>
											</div>
											<div class="modal-footer">
												<a href="Dissertation.html" target="_blank">
													<button type="button" class="btn btn-sm btn-default" >Full Dissertation</button> </a>

												<button type="button" class="btn btn-sm btn-default" data-dismiss="modal">Close</button>
											</div>
										</div>
									</div>
								</div>
								<!-- Modal end -->
							</div>


							<div class="col-sm-6 col-md-3 isotope-item Metabolomics Methodology">
								<div class="image-box">
									<div class="overlay-container">
										<img src="images/portfolio-1.png" alt="">
										<a class="overlay" data-toggle="modal" data-target="#project-1">
											<i class="fa fa-search-plus"></i>
											<a class="btn btn-default btn-block" data-toggle="modal" data-target="#project-1">Published</a>
										</a>
									</div>
									
									<h6 class="text-center">A Workflow for Missing Values Imputation of Untargeted Metabolomics Data<br><br></h6> 
								</div>
								<!-- Modal -->
								<div class="modal fade" id="project-1" tabindex="-1" role="dialog" aria-labelledby="project-1-label" aria-hidden="true">
									<div class="modal-dialog modal-lg">
										<div class="modal-content">
											<div class="modal-header">
												<button type="button" class="close" data-dismiss="modal"><span aria-hidden="true">&times;</span><span class="sr-only">Close</span></button>
												<h4 class="modal-title" id="project-1-label">
													A Workflow for Missing Values Imputation of Untargeted Metabolomics Data (2020)</h4>
											</div>
											<div class="modal-body">
												
												<div class="row">												 
													<div class="col-md-12">
														<h7><i>Faquih T, van Smeden M, Luo J, le Cessie S, Kastenmüller G, Krumsiek J, Noordam R, van Heemst D, Rosendaal FR, van Hylckama Vlieg A, Willems van Dijk K, Mook-Kanamori DO. A Workflow for Missing Values Imputation of Untargeted Metabolomics Data. Metabolites. 2020 Nov 26;10(12):486. doi: 10.3390/metabo10120486. PMID: 33256233; PMCID: PMC7761057.</i>
														</h7><br>
														<p><b>Abstract</b><br>
															Metabolomics studies have seen a steady growth due to the development and implementation of affordable and high-quality metabolomics platforms. In large metabolite panels, measurement values are frequently missing and, if neglected or sub-optimally imputed, can cause biased study results. We provided a publicly available, user-friendly R script to streamline the imputation of missing endogenous, unannotated, and xenobiotic metabolites. We evaluated the multivariate imputation by chained equations (MICE) and k-nearest neighbors (kNN) analyses implemented in our script by simulations using measured metabolites data from the Netherlands Epidemiology of Obesity (NEO) study (n = 599). We simulated missing values in four unique metabolites from different pathways with different correlation structures in three sample sizes (599, 150, 50) with three missing percentages (15%, 30%, 60%), and using two missing mechanisms (completely at random and not at random). Based on the simulations, we found that for MICE, larger sample size was the primary factor decreasing bias and error. For kNN, the primary factor reducing bias and error was the metabolite correlation with its predictor metabolites. MICE provided consistently higher performance measures particularly for larger datasets (n > 50). In conclusion, we presented an imputation workflow in a publicly available R script to impute untargeted metabolomics data. Our simulations provided insight into the effects of sample size, percentage missing, and correlation structure on the accuracy of the two imputation methods.
															<br><br>
															<b>Keywords:</b> imputation; k-nearest neighbors; metabolon; multiple imputation using chained equations; simulation; untargeted metabolomics; workflow. 
															
														</p>
													</div>
												</div>
											</div>
											<div class="modal-footer">
												<a href="https://doi.org/10.3390/metabo10120486" target="_blank">
													<button type="button" class="btn btn-sm btn-default" >Full Text</button> </a>

												<a href="https://pubmed.ncbi.nlm.nih.gov/33256233/" target="_blank">
												<button type="button" class="btn btn-sm btn-default" >PubMed</button> </a>

												<a href="https://github.com/tofaquih/imputation_of_untargeted_metabolites/tree/v1.3" target="_blank">
													<button type="button" class="btn btn-sm btn-default" >GitHub Repo</button> </a>

												<button type="button" class="btn btn-sm btn-default" data-dismiss="modal">Close</button>
											</div>
										</div>
									</div>
								</div>
								<!-- Modal end -->
							</div>




							<div class="col-sm-6 col-md-3 isotope-item Proteomics Methodology">
								<div class="image-box">
									<div class="overlay-container">
										<img src="images/portfolio-2.png" alt="">
										<a class="overlay" data-toggle="modal" data-target="#project-2">
											<i class="fa fa-search-plus"></i>
											 
										</a>
									</div>
									<a class="btn btn-default btn-block" data-toggle="modal" data-target="#project-2">Published</a>
								</div>
								<h6 class="text-center">Agreement of aptamer proteomics with standard methods for measuring venous thrombosis biomarkers</h6>

								<!-- Modal -->
								<div class="modal fade" id="project-2" tabindex="-1" role="dialog" aria-labelledby="project-2-label" aria-hidden="true">
									<div class="modal-dialog modal-lg">
										<div class="modal-content">
											<div class="modal-header">
												<button type="button" class="close" data-dismiss="modal"><span aria-hidden="true">&times;</span><span class="sr-only">Close</span></button>
												<h4 class="modal-title" id="project-2-label">Agreement of aptamer proteomics with standard methods for measuring venous thrombosis biomarkers (2021)
												</h4>
											</div>
											<div class="modal-body">
												
												<div class="row">												 
													<div class="col-md-12">
														<h7><i>Faquih T, Mook-Kanamori DO, Rosendaal FR, Baglin T, Willems van Dijk K, van Hylckama Vlieg A. Agreement of aptamer proteomics with standard methods for measuring venous thrombosis biomarkers. Res Pract Thromb Haemost. 2021 May 4;5(4):e12526. doi: 10.1002/rth2.12526. PMID: 34013156; PMCID: PMC8110437.</i>
														</h7><br>
														<p><b>Abstract</b><br>
															Background: Venous thromboembolism (VTE) is a complex disease with an incidence rate of about 1 in 1000 per year. Despite the availability of validated biomarkers for VTE, unprovoked events account for 50% of first events. Therefore, emerging high-throughput proteomics are promising methods for the expansion of VTE biomarkers. One such promising high-throughput platform is SomaScan, which uses a large library of synthetic oligonucleotide ligands known as aptamers to measure thousands of proteins.
	
															Objective: The aim of this study was to evaluate the viability of the aptamer-based SomaScan platform for VTE studies by examining its agreement with standard laboratory methods.
	
															Methods: We examined the agreement between eight established VTE biomarkers measured by SomaScan and standard laboratory immunoassay and viscosity-based instruments in 54 individuals (27 cases and 27 controls) from the Thrombophilia, Hypercoagulability and Environmental Risks in Venous Thromboembolism study. We performed the agreement analysis by using a regression model and predicting the estimates and the 95% prediction interval (PI) of the laboratory instrument values using SomaScan values.
	
															Results: SomaScan measurements exhibited overall poor agreement, particularly for D-dimer (average fit, 492.7 ng/mL; 95% PI, 110.0-1998.2) and fibrinogen (average fit, 3.3 g/L; 95% PI, 2.0-4.7).
	
															Conclusion: Our results indicate that SomaScan measurement had poor agreement with the standard laboratory measurements. These results may explain why some genome-wide association studies with VTE proteins measured by SomaScan did not confirm previously identified loci. Therefore, SomaScan should be considered with caution in VTE studies.
															<br><br>
															<b>Keywords:</b> aptamers; biomarkers; blood coagulation factors; immunoassay; nucleotide; proteomics; venous thrombosis.</p>
													</div>
												</div>
											</div>
											<div class="modal-footer">
												<a href="https://doi.org/10.1002/rth2.12526" target="_blank">
													<button type="button" class="btn btn-sm btn-default" >Full Text</button> </a>

												<a href="https://pubmed.ncbi.nlm.nih.gov/34013156/" target="_blank">
												<button type="button" class="btn btn-sm btn-default" >PubMed</button> </a>

												<button type="button" class="btn btn-sm btn-default" data-dismiss="modal">Close</button>
											</div>
										</div>
									</div>
								</div>
								<!-- Modal end -->
							</div>
							

							
							<div class="col-sm-6 col-md-3 isotope-item Etiology Metabolomics">
								<div class="image-box">
									<div class="overlay-container">
										<img src="images/portfolio-3.png" alt="">
										<a class="overlay" data-toggle="modal" data-target="#project-3">
											<i class="fa fa-search-plus"></i> 
										</a>
									</div>
									<a class="btn btn-default btn-block" data-toggle="modal" data-target="#project-3">In Progress</a>
								</div>
								<h6 class="text-center">Atlas of metabolites and pathways associated with hepatic triglyceride content in middle aged men and women</h6>

								<!-- Modal -->
								<div class="modal fade" id="project-3" tabindex="-1" role="dialog" aria-labelledby="project-3-label" aria-hidden="true">
									<div class="modal-dialog modal-lg">
										<div class="modal-content">
											<div class="modal-header">
												<button type="button" class="close" data-dismiss="modal"><span aria-hidden="true">&times;</span><span class="sr-only">Close</span></button>
												<h4 class="modal-title" id="project-3-label">Atlas of metabolites and pathways associated with hepatic triglyceride content in middle aged men and women.</h4>
											</div>
											<div class="modal-body">
												
												<div class="row">												 
													<div class="col-md-12">
														<h7><i></i>
														</h7><br>
														<p><b>Abstract</b><br>
															Background Non-alcoholic fatty liver disease (NAFLD) is characterized by the pathological accumulation of triglycerides in hepatocytes and is associated with insulin resistance and atherogenic dyslipidemia. Thus far, the extent of metabolic dysregulation associated with hepatic triglyceride accumulation has not been fully addressed.
															<br><br>
															<b>Keywords:</b> </p>
													</div>
												</div>
											</div>
											<div class="modal-footer">

												<a href="https://tofaquih.github.io/AtlasLiver/" target="_blank">
												<button type="button" class="btn btn-sm btn-default" >Website and Networks</button> </a>

												<button type="button" class="btn btn-sm btn-default" data-dismiss="modal">Close</button>
											</div>
										</div>
									</div>
								</div>
								<!-- Modal end -->
							</div>

							<!--CAG-->
							<div class="col-sm-6 col-md-3 isotope-item Etiology Metabolomics Genomics">
								<div class="image-box">
									<div class="overlay-container">
										<img src="images/portfolio-6.png" alt="">
										<a class="overlay" data-toggle="modal" data-target="#project-6">
											<i class="fa fa-search-plus"></i> 
										</a>
									</div>
									<a class="btn btn-default btn-block" data-toggle="modal" data-target="#project-6">Published</a>
								</div>
								<h6 class="text-center">Normal range CAG repeat size variations in the HTT gene and their assoction with lipoproteins profile</h6>

								<!-- Modal -->
								<div class="modal fade" id="project-6" tabindex="-1" role="dialog" aria-labelledby="project-6-label" aria-hidden="true">
									<div class="modal-dialog modal-lg">
										<div class="modal-content">
											<div class="modal-header">
												<button type="button" class="close" data-dismiss="modal"><span aria-hidden="true">&times;</span><span class="sr-only">Close</span></button>
												<h4 class="modal-title" id="project-6-label">Normal range CAG repeat size variations in the HTT gene are associated with an adverse lipoprotein profile partially mediated by body mass index</h4>
											</div>
											<div class="modal-body">
												
												<div class="row">												 
													<div class="col-md-12">
														<h7><i>

															Tariq O Faquih, N Ahmad Aziz, Sarah L Gardiner, Ruifang Li-Gao, Renée de Mutsert, Yuri Milaneschi, Stella Trompet, J Wouter Jukema, Frits R Rosendaal, Astrid Hylckama Vlieg, Ko Willems Dijk, Dennis O Mook-Kanamori, Normal range CAG repeat size variations in the HTT gene are associated with an adverse lipoprotein profile partially mediated by body mass index, Human Molecular Genetics, 2023;, ddad020, https://doi.org/10.1093/hmg/ddad020 </i>
														</h7><br>
														<p><b>Abstract</b><br>
															Tandem CAG repeat sizes of 36 or more in the huntingtin gene (HTT) cause Huntington disease. Apart from neuropsychiatric complications, the disease is also accompanied by metabolic dysregulation and weight loss, which contribute to a progressive functional decline. Recent studies also reported an association between repeats below the pathogenic threshold (<36) for Huntington’s disease and body mass index (BMI), suggesting that HTT repeat sizes in the non-pathogenic range are associated with metabolic dysregulation. 
															In this study we hypothesized that HTT repeat sizes <36 are associated with metabolite levels, possibly mediated through reduced BMI. We pooled data from three European cohorts (n=10,228) with genotyped HTT CAG repeat size and metabolomic measurements. All 145 metabolites were measured on the same targeted platform in all studies. Multilevel mixed-effects analysis using the CAG repeat size in HTT identified 67 repeat size-metabolite associations. Overall, the metabolomic profile associated with larger CAG repeat sizes in HTT were unfavorable—similar to those of higher risk of coronary artery disease and type 2 diabetes—and included elevated levels of amino acids, fatty acids, LDL, VLDL and IDL related metabolites whilst with decreased levels of very large HDL related metabolites. Furthermore, the associations of 50 metabolites, in particular specific very large HDL related metabolites, were mediated by lower BMI. However, no mediation effect was found for 17 metabolites related to LDL and IDL. In conclusion, our findings indicate that large non-pathogenic CAG repeat sizes in HTT are associated with an unfavorable metabolomic profile despite their association with a lower BMI.
															<br><br>
															<b>Keywords:</b> Huntingtin gene, CAG repeats, Lipoproteins, Metabolomics, Multilevel mixed-effects analysis </p>
													</div>
												</div>
											</div>
											<div class="modal-footer">
												<a href="https://academic.oup.com/hmg/advance-article/doi/10.1093/hmg/ddad020/7009474" target="_blank">
													<button type="button" class="btn btn-sm btn-default" >Full Text</button> </a>

												<a href="https://pubmed.ncbi.nlm.nih.gov/36715614/" target="_blank">
													<button type="button" class="btn btn-sm btn-default" >PubMed</button> </a>

												<button type="button" class="btn btn-sm btn-default" data-dismiss="modal">Close</button>
											</div>
										</div>
									</div>
								</div>
								<!-- Modal end -->
							</div>
						
							
							<!--PFAS-->
							
							<div class="col-sm-6 col-md-3 isotope-item Etiology Metabolomics" style="top: 300px">
								<div class="image-box">
									<div class="overlay-container">
										<img src="images/portfolio-5.png" alt="">
										<a class="overlay" data-toggle="modal" data-target="#project-5">
											<i class="fa fa-search-plus"></i> 
										</a>
									</div>
									<a class="btn btn-default btn-block" data-toggle="modal" data-target="#project-5">In Progress</a>
								</div>
								<h6 class="text-center">PFAS concentrations are associated with a cardio-metabolic risk profile: findings from two population cohorts</h6>

								<!-- Modal -->
								<div class="modal fade" id="project-5" tabindex="-1" role="dialog" aria-labelledby="project-5-label" aria-hidden="true">
									<div class="modal-dialog modal-lg">
										<div class="modal-content">
											<div class="modal-header">
												<button type="button" class="close" data-dismiss="modal"><span aria-hidden="true">&times;</span><span class="sr-only">Close</span></button>
												<h4 class="modal-title" id="project-5-label">PFAS concentrations are associated with a cardio-metabolic risk profile: findings from two population cohorts</h4>
											</div>
											<div class="modal-body">
												
												<div class="row">												 
													<div class="col-md-12">
														<h7><i></i>
														</h7><br>
														<p><b>Abstract</b><br>
															Coming Soon
															<br><br>
															<b>Keywords:</b> </p>
													</div>
												</div>
											</div>
											<div class="modal-footer">

												<button type="button" class="btn btn-sm btn-default" data-dismiss="modal">Close</button>
											</div>
										</div>
									</div>
								</div>
								<!-- Modal end -->
							</div>


							<!--MAGE-->
							
							<div class="col-sm-6 col-md-3 isotope-item Prediction Metabolomics" style="top: 300px">
								<div class="image-box">
									<div class="overlay-container">
										<img src="images/portfolio-7.png" alt="">
										<a class="overlay" data-toggle="modal" data-target="#project-7">
											<i class="fa fa-search-plus"></i> 
										</a>
									</div>
									<a class="btn btn-default btn-block" data-toggle="modal" data-target="#project-7">In Progress</a>
								</div>
								<h6 class="text-center">Robust Metabolomic Age Prediction Based on a Wide Selection of Metabolites</h6>

								<!-- Modal -->
								<div class="modal fade" id="project-7" tabindex="-1" role="dialog" aria-labelledby="project-7-label" aria-hidden="true">
									<div class="modal-dialog modal-lg">
										<div class="modal-content">
											<div class="modal-header">
												<button type="button" class="close" data-dismiss="modal"><span aria-hidden="true">&times;</span><span class="sr-only">Close</span></button>
												<h4 class="modal-title" id="project-7-label">Robust Metabolomic Age Prediction Based on a Wide Selection of Metabolites</h4>
											</div>
											<div class="modal-body">
												
												<div class="row">												 
													<div class="col-md-12">
														<h7><i></i>
														</h7><br>
														<p><b>Abstract</b><br>
															Coming Soon
															<br><br>
															<b>Keywords:</b> </p>
													</div>
												</div>
											</div>
											<div class="modal-footer">

												<button type="button" class="btn btn-sm btn-default" data-dismiss="modal">Close</button>
											</div>
										</div>
									</div>
								</div>
								<!-- Modal end -->
							</div>



 							<div class="col-sm-6 col-md-3 isotope-item Genomics" >
								<div class="image-box">
									<div class="overlay-container">
										<img src="images/portfolio-4.png" alt="">
										<a class="overlay" data-toggle="modal" data-target="#project-4">
											<i class="fa fa-search-plus"></i> 
										</a>
									</div>
									<a class="btn btn-default btn-block" data-toggle="modal" data-target="#project-4">Published</a>
								</div>
								<h6 class="text-center">Saudi Human Genome Project</h6>

								<!-- Modal -->
								<div class="modal fade" id="project-4" tabindex="-1" role="dialog" aria-labelledby="project-4-label" aria-hidden="true">
									<div class="modal-dialog modal-lg">
										<div class="modal-content">
											<div class="modal-header">
												<button type="button" class="close" data-dismiss="modal"><span aria-hidden="true">&times;</span><span class="sr-only">Close</span></button>
												<h4 class="modal-title" id="project-4-label">Saudi Human Genome Project (2013-2019)</h4>
											</div>
											<div class="modal-body">
												
												<div class="row">												 
													<div class="col-md-12">
														<h3>Project Description</h3>
														<p>During my time in at King Faisal Specialist Hospital and Research Center (Saudi Arabia) I Worked as bioinformatician as part of the Saudi Human Genome Project. The goal of the project was to "To determine the variants underlying genetic disease in the Kingdom of Saudi Arabia, localize and develop genetic sequencing technologies, the field of genomics and bioinformatics in order to lay the foundation of personalized medicine". <br>
															My work involved using and developing tools for exome and full genome analysis and annotation and database development and management. 
															
															<br/><br/>

															<b>Notable Publications:</b>

															<br><br><h7><i>Monies D, et al. The landscape of genetic diseases in Saudi Arabia based on the first 1000 diagnostic panels and exomes. Hum Genet. 2017 Aug;136(8):921-939. doi: 10.1007/s00439-017-1821-8. Epub 2017 Jun 9. PMID: 28600779; PMCID: PMC5502059.</i></h7> 

															<br><br><h7><i>Yemni EA, et al. Integrated Analysis of Whole Exome Sequencing and Copy Number Evaluation in Parkinson's Disease. Sci Rep. 2019 Mar 4;9(1):3344. doi: 10.1038/s41598-019-40102-x. PMID: 30833663; PMCID: PMC6399448.</i></h7> 

															<br><br><h7><i>Abouelhoda M, Faquih T, El-Kalioby M, Alkuraya FS. Revisiting the morbid genome of Mendelian disorders. Genome Biol. 2016 Nov 24;17(1):235. doi: 10.1186/s13059-016-1102-1. PMID: 27884173; PMCID: PMC5123336.</i></h7> 
															</p>
													</div>
												</div>
											</div>
											<div class="modal-footer">
												<button type="button" class="btn btn-sm btn-default" data-dismiss="modal">Close</button>
											</div>
										</div>
									</div>
								</div>
								<!-- Modal end -->
							</div> 
							


						</div>
						<!-- portfolio items end -->
					
					</div>
				</div>
			</div>
		</section>
		<!-- section end -->

		<div class="default-bg colord secPadding">
			<div class="container">

				</div>
			</div>
		</div>


		<!-- footer start --> 
		<footer id="footer">
<!-- .footer start --> 

<!-- .footer end -->
			<!-- .subfooter start --> 
			<div class="subfooter">
				<div class="container">
					<div class="row">
						
						<div class="col-md-12">
							
							<p class="text-center">Copyright © 2023 Tariq Faquih.</p>
						</div>
					</div>
				</div>
			</div>
			<!-- .subfooter end -->

		</footer>
		<!-- footer end -->

		<!-- JavaScript --> 
		<script type="text/javascript" src="plugins/jquery.min.js"></script>
		<script type="text/javascript" src="bootstrap/js/bootstrap.min.js"></script>
		<script type="text/javascript" src="plugins/modernizr.js"></script>
		<script type="text/javascript" src="plugins/isotope/isotope.pkgd.min.js"></script>
		<script type="text/javascript" src="plugins/jquery.backstretch.min.js"></script>
		<script type="text/javascript" src="plugins/jquery.appear.js"></script>

		<!-- Custom Scripts -->
		<script type="text/javascript" src="js/custom.js"></script>

	</body>
</html>
