<!DOCTYPE html>
<!--[if IE 9]> <html lang="en" class="ie9"> <![endif]-->
<!--[if !IE]><!-->
<html lang="en">
<!--<![endif]-->
	<head>
		<meta name="google-site-verification" content="bzK5FySMBHqsDWYaHw6KeeAYY11QrOz0gUyP8iPx3Uw" />
		<meta charset="utf-8">
		<title>Tariq Faquih's Academic Webpage</title>
		<meta name="description" content="My personal webpage">
		<meta name="Tariq Faquih" content="">

		<!-- Mobile Meta -->
		<meta name="viewport" content="width=device-width, initial-scale=1.0">

		<!-- Favicon -->
		<link rel="shortcut icon" href="images/favicon.ico">
 
		<link href="./bootstrap/css/bootstrap.css" rel="stylesheet" type="text/css">  
		<link href="./fonts/font-awesome/css/font-awesome.css" rel="stylesheet" type="text/css"> 
		<link href="./fonts/academicons-1.9.1/css/academicons.css" rel="stylesheet" type="text/css"> 
		<link href="./css/animations.css" rel="stylesheet" type="text/css"> 
		<link href="./css/style.css" rel="stylesheet" type="text/css"> 
		<link href="./css/custom.css" rel="stylesheet" type="text/css">
		<!-- <link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro:300,400,700,300italic,400italic,700italic" rel="stylesheet"> -->

	</head>

	<body class="no-trans">
		<!-- scrollToTop --> 
		<div class="scrollToTop"><i class="icon-up-open-big"></i></div>

		<!-- header start --> 
		<header class="header fixed clearfix navbar navbar-fixed-top">
			<div class="container">
				<div class="row">
					<div class="col-md-4">

						<!-- header-left start --> 
						<div class="header-left">

							<!-- logo -->
							<div class="logo smooth-scroll">
								<a href="#banner"><img id="logo" src="images/logo2.png" alt="Worthy"></a>	
							</div>

							<!-- name-and-slogan -->
							<div class="logo-section smooth-scroll">
								<div class="brand"><a href="#banner">Tariq Faquih</a></div>								
							</div>

						</div>
						<!-- header-left end -->

					</div>
					<div class="col-md-8">

						<!-- header-right start --> 
						<div class="header-right">

							<!-- main-navigation start --> 
							<div class="main-navigation animated">

								<!-- navbar start --> 
								<nav class="navbar navbar-default" role="navigation">
									<div class="container-fluid">

										<!-- Toggle get grouped for better mobile display -->
										<div class="navbar-header">
											<button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#navbar-collapse-1">
												<span class="sr-only">Toggle navigation</span>
												<span class="icon-bar"></span>
												<span class="icon-bar"></span>
												<span class="icon-bar"></span>
											</button>
										</div>

										<!-- Collect the nav links, forms, and other content for toggling -->
										<div class="collapse navbar-collapse scrollspy smooth-scroll" id="navbar-collapse-1">
											<ul class="nav navbar-nav navbar-right">
												<li class="active"><a href="#banner">Home</a></li>
												<li><a href="#about">CV</a></li>
												<li><a href="#portfolio">Portfolio</a></li>
												<li><a href="Dissertation.html">Dissertation</a></li>
												

											</ul>
										</div>

									</div>
								</nav>
								<!-- navbar end -->

							</div>
							<!-- main-navigation end -->

						</div>
						<!-- header-right end -->

					</div>
				</div>
			</div>
		</header>
		<!-- header end -->

		<!-- banner start --> 
		<div id="banner" class="banner">
			<div class="banner-image"></div>
			<div class="banner-caption">
				<div class="container">
					<div class="row">					 
						<div class="caption-data" style="margin-top: 0px; opacity: 1;" data-animation-effect="fadeIn">
								<h1>Hello!<br>My Name is Tariq Faquih</h1>
								<p class="lead text-center">I am an Epidemiologist and Bioinformatician specialized in Genomics and Metabolomics. 
									<br>Welcome to my Academic Webpage.</p>
								<div class="padding-top60 contact-form">
								
									<ul class="social-links"  >
										<li class="github"><a target="_blank" href="https://github.com/tofaquih"><i class="fa fa-github"></i></a></li>
										<li class="twitter"><a target="_blank" href="https://twitter.com/TariqFaquih"><i class="fa fa-twitter"></i></a></li>
										<li class="google-scholar"><a target="_blank" href="https://scholar.google.nl/citations?hl=en&user=wYInss0AAAAJ"><i class="ai ai-google-scholar"></i></a></li>
										<li class="pubmed"><a target="_blank" href="https://pubmed.ncbi.nlm.nih.gov/?term=tariq+faquih&sort=date"><i class="ai ai-pubmed"></i></a></li>
										<li class="linkedin"><a target="_blank" href="https://www.linkedin.com/in/tariq-faquih-0b434782/"><i class="fa fa-linkedin"></i></a></li>
										<li class="orcid"><a target="_blank" href="https://orcid.org/0000-0001-8026-2251"><i class="ai ai-orcid"></i></a></li>
									</ul>
									
									

								</div>
							</div>
					</div>
				</div>
			</div>
		</div>
		<!-- banner end -->


		

 
		<!-- section start --> 
		<section class="section clearfix no-view secPadding" data-animation-effect="fadeIn">
			<div class="container">
				<div class="row">
					<div class="col-md-12">
						<h1 id="about" class="title text-center">About <span>Me</span></h1>
						<p class="lead text-center">I am a currently a Research Fellow at the Sleep Medicine Epidemiology Division at Brigham and Women's Hospital<br>
							<br>My focus is on epidemiological studies and methodologies using genomics, metabolomics, and proteomics.</p>
						<div class="space"></div>
						<div class="row">
							<div class="col-md-6">
								<img src="./images/port 1 .jpg" alt="">
								<div class="space"></div>
							</div>
							<div class="col-md-6">
								<p><b>My current research at the Sleep Medicine Division at Brigham and Women's Hospital and Harvard Medical School focuses on utilizing Omics for epidemiological studies on sleep disorders and cardiometabolic outcomes associated with sleep disorders.</b></p>
								<p>I have secondary affiliations with Sofer Lab (BIDMC) and Saxena Lab (Broad Institute)</p>
								<p>I received a scholarship for my PhD candidacy at the Department of Clinical Epidemiology, LUMC, Netherlands. <b>My PhD thesis, <u>"Multi-Omics in Research: Epidemiology, Methodology, and Advanced Data Analysis"</u></b>, focuses on using multiple Omics to study a variety of ailments. These Omics include genomics (Genome Wide Association Studies, Genomic Risk Scores), Metabolomics (large data from UHPLC-MS/MS and NMR platforms), and proteomics.</p>
								<p></p>
								<ul class="list-unstyled">
									<li><i class="fa fa-arrow-circle-right pr-10 colored"></i> Worked as a visiting researcher at the University of Cambridge (Department of Public Health and Primary care). It was a cooperative effort to perform an analysis for one of my ongoing projects on “metabolomics aging and disease prediction”.</li>
									<li><i class="fa fa-arrow-circle-right pr-10 colored"></i> Projects involves collbrations with TOPMed, UK Biobank, All of Us, and the National Sleep Research Resource</li>
									<li><i class="fa fa-arrow-circle-right pr-10 colored"></i> Besides our direct projects we are also working on secondary projects with The Cohorts for Heart and Aging Research in Genomic Epidemiology (CHARGE) Consortium.</li>
									<li><i class="fa fa-arrow-circle-right pr-10 colored"></i> Organized and prioritized work to complete assignments in a timely, efficient manner.</li>
									<li><i class="fa fa-arrow-circle-right pr-10 colored"></i> Experienced in teaching courses for medical students. Courses include basic epidemiology, study designs, and mendelian randomization.</li>
									<a href="pdfs/Faquih, Tariq HMS CV 16032025_021541_v2.pdf" target="_blank">
										<button type="button" class="btn btn-sm btn-default" >View Curriculum Vitae (03/2025)</button> </a>
								</ul>
							</div>
						</div>
						<div class="space"></div>
						<h2>Career and Education</h2>

						<div class="row">
							<div class="col-md-6">
									<ul class="list-unstyled">
										<p>From Saudi Arabia to England, to the United States, and to the Netherlands</p>
									<li><i class="fa fa-arrow-circle-right pr-10 colored"></i> 01/2023- Present : Research Fellow at Brigham Women's Hospital & Harvard Medical School</li>
									<li><i class="fa fa-arrow-circle-right pr-10 colored"></i> 08/2018- 03/2023 : PhD candidate at Leiden University Medical Center</li>
									<li><i class="fa fa-arrow-circle-right pr-10 colored"></i> 10/2013 - 08/2018: Bioinformatician • King Faisal Specialist Hospital and Research Center</li>
									<li><i class="fa fa-arrow-circle-right pr-10 colored"></i> 01/2012 - 06/2013: MSc Bioinformatics • 2013 • Georgetown University, Washington, D.C., USA</li>
									<li><i class="fa fa-arrow-circle-right pr-10 colored"></i> 09/2007 – 07/2011: BSc Human Genetics • 2011 • University of Leeds, Leeds, UK</li>

								</ul>
							</div>
							<div class="col-md-6">
								<div class="panel-group" id="accordion" role="tablist" aria-multiselectable="true">
									<div class="panel panel-default">
										<div class="panel-heading" role="tab" id="headingOne">
											<h4 class="panel-title">
												<a data-toggle="collapse" data-parent="#accordion" href="#collapseOne" aria-expanded="true" aria-controls="collapseOne">
													Personal Skills
												</a>
											</h4>
										</div>
										<div id="collapseOne" class="panel-collapse collapse in" role="tabpanel" aria-labelledby="headingOne">
											<div class="panel-body">
											<li>International experience and network</li>
											<li>Adaptability</li>
											<li>Fast Learner</li>
											<li>Critical thinking and problem solving</li>
											<li>Good teamwork, communication skills and ability to participate in research communities across organizational units.</li>
											</div>
										</div>
									</div>
									<div class="panel panel-default">
										<div class="panel-heading" role="tab" id="headingTwo">
											<h4 class="panel-title">
												<a class="collapsed" data-toggle="collapse" data-parent="#accordion" href="#collapseTwo" aria-expanded="false" aria-controls="collapseTwo">
													Epidemiology and Omics
												</a>
											</h4>
										</div>
										<div id="collapseTwo" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingTwo">
											<div class="panel-body">
												<li>Background in Epidemiology on cross sectional and longitudinal cohorts</li>
												<li>Background in computer science and bioinformatics</li>
												<li>Worked with genomic, proteomics, and metabolomics</li>
												<li>Experienced in Big Data Analysis</li>
												<li>Experienced in Missing Data Analysis</li>
												<li>Backgound in Genetics and working in the lab</li>
											</div>
										</div>
									</div>
									<div class="panel panel-default">
										<div class="panel-heading" role="tab" id="headingThree">
											<h4 class="panel-title">
												<a class="collapsed" data-toggle="collapse" data-parent="#accordion" href="#collapseThree" aria-expanded="false" aria-controls="collapseThree">
													Programming Experience
												</a>
											</h4>
										</div>
										<div id="collapseThree" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingThree">
											<div class="panel-body">
												<li>Proficient in Python and R programming languages</li>
												<li>Worked using Jupyter Notebook and R Studio</li>
												<li>Worked with STATA and SPSS</li>
												<li>Used several HPC clusters for big data analysis</li>
												<li>Worked with database resources such as SQL and Django</li>
												<li>Familiar with Git/GitHub, HTML5, Bash, Adobe Photoshop, Adobe Illustrator and Blender</li>
												<li>Experienced with Windows, MacOS and UNIX operating systems </li>

											</div>
										</div>
									</div>
								</div>
							</div>
						</div>


						
					</div>
				</div>
			</div>
		</section>
		<!-- section end -->

		<!-- section start --> 
		<div class="default-bg colord secPadding">
			<div class="container">

				</div>
			</div>
		</div>
		<!-- section end -->


		<!-- section start --> 
		<section class="section secPadding">
			<div class="container">
				<h1 class="text-center title" id="portfolio">Portfolio</h1>
				<div class="separator"></div>
				<p class="lead text-center">My projects revolve around the usage Multi-Omics to study a variety of phenotypes<br> 
				Here you can find projects on different topics that involved genomcis, proteomics, and metabolomics (and more).</p>
				<br>			
				<div class="row no-view" data-animation-effect="fadeIn">
					<div class="col-md-12">

						<!-- isotope filters start -->
						<div class="filters text-center">
							<ul class="nav nav-pills">
								<li class="active"><a href="#" data-filter="*">All</a></li>
								<li><a href="#" data-filter=".Sleep">Sleep</a></li>
								<li><a href="#" data-filter=".Genomics">Genomics</a></li>
								<li><a href="#" data-filter=".Metabolomics">Metabolomics</a></li>
								<li><a href="#" data-filter=".Proteomics">Proteomics</a></li>
								<li><a href="#" data-filter=".Methodology">Methodology</a></li>
								<li><a href="#" data-filter=".Etiology">Etiology</a></li>
								<li><a href="#" data-filter=".Prediction">Prediction</a></li>
								<li><a href="#" data-filter=".MR">Mendelian Randomization</a></li>
								<li><a href="#" data-filter=".AudioSum">Audio Summary</a></li>
								
							</ul>
						</div>
						<!-- isotope filters end -->


						<!-- portfolio items start -->



						
						<div class="isotope-container row grid-space-20">
							<div class="col-sm-6 col-md-3 isotope-item Metabolomics Methodology Genomics Proteomics Etiology Prediction">
								<div class="image-box">
									<div class="overlay-container">
										<img src="images/dissertation_thumb.png" alt="">
										<a class="overlay" data-toggle="modal" data-target="#project-0">
											<i class="fa fa-search-plus"></i>
											<a class="btn btn-default btn-block" data-toggle="modal" data-target="#project-0">Published</a>
										</a>
									</div>
									
									<h6 class="text-center">Multi-Omics in Research: Epidemiology, Methodology, and Advanced Data Analysis</h6> 
								</div>
								<!-- Modal -->
								<div class="modal fade" id="project-0" tabindex="-1" role="dialog" aria-labelledby="project-0-label" aria-hidden="true">
									<div class="modal-dialog modal-lg">
										<div class="modal-content">
											<div class="modal-header">
												<button type="button" class="close" data-dismiss="modal"><span aria-hidden="true">&times;</span><span class="sr-only">Close</span></button>
												<h4 class="modal-title" id="project-0-label">
													PhD Dissertation: Multi-Omics in Research: Epidemiology, Methodology, and Advanced Data Analysis (2023)</h4>
											</div>
											<div class="modal-body">
												
												<div class="row">												 
													<div class="col-md-12">
														<h7><i>PhD Dissertation: Multi-Omics in Research: Epidemiology, Methodology, and Advanced Data Analysis (2023)</i>
														</h7><br>
														<p><b>Summary</b><br>
															Metabolomics, proteomics, and genomics analyses provide profound insight into human biology
															and disease pathophysiology In this thesis, we explored the methodological challenges facing
															these OMICs technologies and illustrated their applications in epidemiological studies. In part
															one, we focused on some of the methodological challenges facing OMICs research. Chapter 2
															examined the agreement between aptamer-based proteomics measurements of venous thrombosis
															(VTE) biomarkers with those measured by standardized clinical instruments. We reported
															that 9 venous thrombosis biomarkers showed poor agreement with the clinical measurements.
															The agreement was particularly poor for D-dimer, a marker often used in VTE diagnosis. In
															Chapter 3, we explored the challenges of handling missing values in metabolomics data, particularly
															from untargeted metabolomics platforms. We assessed the performance of two previously
															reported methods for imputation, namely “k-nearest neighbor” (kNN) and “multiple imputation
															using chained equations”, by simulating missing patterns in data from the NEO study. Our
															findings showed that the multiple imputation method had less bias than the kNN method in
															most scenarios, except for the scenario with small sample size and high level of missingness.
															We further provided a publicly available R script to streamline the process of imputing missing
															metabolomics data using the two methods.
															Metabolites can provide unique insight into biological pathophysiology as they encompass the
															cumulative effects of genetic, lifestyle, and environmental factors. Thus, metabolomics analyses
															have been used to study biological processes such as aging. Some studies have used metabolomics
															to estimate “metabolomic age”, a reflection of aging on a metabolomic level, using
															prediction modelling methods. In Chapter 4, we addressed common challenges for developing
															metabolomics-based age prediction models and developed a model to predict metabolomic
															age using the Metabolon metabolomics platform measurements. We developed a model in the
															INTERVAL study using ridge regression and bootstrapping. This study population is a relatively
															healthy large population (n = 11,977) with a wide age range. Overall, after development and
															internal validation, our prediction models for metabolomic age demonstrated good performance
															(adjusted R2 = 0.83).
															The second part of the thesis addressed various epidemiological research questions by utilizing
															genomic data and metabolomics measurements (Metabolon and Nightingale platforms) and
															using advanced data analysis methods. In Chapter 5, we examined the associations of over 1300
															metabolites, measured by the Metabolon platform, with hepatic triglycerides content (HTGC).
															These associations included a range of endogenous metabolites from various pathways (particularly
															amino acids and lipids), as well as novel and uncharacterized metabolites. Subsequently,
															we constructed two complementary holistic networks, one biologically based and the other
															statistically driven, to assign the associated metabolites to pathways. These interactive networks
															revealed potential novel pathways associated with HTGC and possibly fatty liver diseases. We
															further provided these networks on an online, publicly available webpage for further exploration
															by the research community.
															One type of variation in the genome is variation in the number of repeated cytosine-adenine-
															guanine nucleotide sequences (CAG repeats). When the size of these repeats is beyond
															specific pathogenic thresholds in or near specific genes, they lead to the onset of several severe
															neurodegenerative diseases. One such case is the large CAG repeat sizes in the huntingtin (HTT)
															gene leading to the onset of Huntington’s disease. Interestingly, recent studies have found that
															CAG repeat sizes even within the non-pathogenic range are associated with health outcomes
															such as cognitive function, depression, and body weight. In Chapter 6, we explored the possible
															association between CAG repeat variation sizes below the pathogenic threshold (<36) in the
															HTT gene and the metabolite levels measured by the Nightingale platform. Multilevel mixed-effects
															and mediation analyses were conducted in pooled data from three large European cohorts
															(n = 10,275). Our results showed that large non-pathogenic CAG repeat sizes were associated
															with an unfavorable metabolomic profile, similar to that for an increased risk of cardiometabolic
															diseases.
															One of the advantages of metabolomics is the capability to measure “external” metabolites,
															often referred to as xenobiotics, from medication use, diet, and environmental exposures and
															contaminations. One such group of measured xenobiotics are the man-made per- and polyfluoroalkyl
															substances (PFAS)—commonly known as “forever chemicals” due to their persistence in
															the environment and the body. In addition to their persistent nature, high concentrations of PFAS
															have been reported to be associated with severe health impact. Despite efforts to regulate their
															production, PFAS levels remain detectable in the soil, drinking water, and human blood samples
															in the Netherlands and Germany. In Chapter 7, we studied the associations between common
															PFAS levels in blood, as measured by the Metabolon platform, and metabolomics and lipoprotein
															profiles (Nightingale platform) of the general population in the Netherlands (NEO study) and
															Germany (Rhineland study). Overall, PFAS metabolites were detectable in nearly all participants
															of both studies. Furthermore, as confirmed by meta-analysis, PFAS levels were associated with an
															unhealthy metabolomic profile characterized by increased fatty acids, low density lipoproteins,
															and apolipoprotein B levels. These associations are reminiscent of the metabolomic profile for
															increased risk of cardiometabolic diseases and were found to be particularly stronger in younger
															individuals. These data indicate that PFAS are a health risk even in individuals that apparently
															have not been exposed to high levels of PFAS.
															<br><br>
															
														</p>
													</div>
												</div>
											</div>
											<div class="modal-footer">
												<a href="Dissertation.html" target="_blank">
													<button type="button" class="btn btn-sm btn-default" >Full Dissertation</button> </a>

												<button type="button" class="btn btn-sm btn-default" data-dismiss="modal">Close</button>
											</div>
										</div>
									</div>
								</div>
								<!-- Modal end -->
							</div>




							<!--CAGsleep-->
							
							<div class="col-sm-6 col-md-3 isotope-item Genomics Sleep Etiology Prediction Metabolomics" style="top: 300px">
								<div class="image-box">
									<div class="overlay-container">
										<img src="images/cagSleep.png" alt="">
										<a class="overlay" data-toggle="modal" data-target="#project-16">
											<i class="fa fa-search-plus"></i>
										</a>
									</div>
									<a class="btn btn-default btn-block" data-toggle="modal" data-target="#project-16">In Progress</a>
								</div>
								<h6 class="text-center">CAG Repeat Variations in the HTT, CACNA1A, and ATXN3 genes and their Implications on Sleep Traits </h6>
							
								<!-- Modal -->
								<div class="modal fade" id="project-16" tabindex="-1" role="dialog" aria-labelledby="project-16-label"
									aria-hidden="true">
									<div class="modal-dialog modal-lg">
										<div class="modal-content">
											<div class="modal-header">
												<button type="button" class="close" data-dismiss="modal"><span
														aria-hidden="true">&times;</span><span class="sr-only">Close</span></button>
												<h4 class="modal-title" id="project-16-label">CAG Repeat Variations in the HTT, CACNA1A, and ATXN3 genes and their Implications on Sleep Traits</h4>
											</div>
											<div class="modal-body">
							
												<div class="row">
													<div class="col-md-12">
														<h7><i>Tariq Faquih, Linjun Ao, Heming Wang, Raymond Noordam et al.</i>
														</h7><br>
														<p><b>Abstract </b><br>

															 <br><br>
																<b>Keywords:</b> tandem repeats; sleep; CAG; depression
														</p>
													</div>
												</div>
											</div>
											<div class="modal-footer">
												<!-- <a href="" target="_blank">
																				<button type="button" class="btn btn-sm btn-default" >PubMed</button>
																				<a href="PodcastPage.html?" target="_blank">
																					<button type="button" class="btn btn-sm btn-default" >AI Podcast</button> -->
												<button type="button" class="btn btn-sm btn-default" data-dismiss="modal">Close</button>
											</div>
										</div>
									</div>
								</div>
								<!-- Modal end -->
							</div>
							
							
							<!--vtesleep-->
							
							<div class="col-sm-6 col-md-3 isotope-item Genomics Sleep MR Proteomics Etiology" style="top: 300px">
								<div class="image-box">
									<div class="overlay-container">
										<img src="images/project-15.jpg" alt="">
										<a class="overlay" data-toggle="modal" data-target="#project-15">
											<i class="fa fa-search-plus"></i>
										</a>
									</div>
									<a class="btn btn-default btn-block" data-toggle="modal" data-target="#project-15">In Progress</a>
								</div>
								<h6 class="text-center">The associations between sleep health and hemostasis factors in
									population-based cohort studies </h6>
							
								<!-- Modal -->
								<div class="modal fade" id="project-15" tabindex="-1" role="dialog" aria-labelledby="project-15-label"
									aria-hidden="true">
									<div class="modal-dialog modal-lg">
										<div class="modal-content">
											<div class="modal-header">
												<button type="button" class="close" data-dismiss="modal"><span
														aria-hidden="true">&times;</span><span class="sr-only">Close</span></button>
												<h4 class="modal-title" id="project-15-label">The associations between sleep health and hemostasis
													factors in
													population-based cohort studies </h4>
											</div>
											<div class="modal-body">
							
												<div class="row">
													<div class="col-md-12">
														<h7><i>T. Faquih, J. Han, R. de Mutsert, R. Noordam, T. Sofer,
																S. Redline, F. Rosendaal, R. Li-Gao, H. Wang and
																A. van Hylckama Vlieg</i>
														</h7><br>
														<p><b>Abstract (ISTH2024)</b><br>
															Background: Sleep disruption has adverse effects on the cardiovascular
															system. Although previous studies suggested the influence of sleep health
															on the activation of hemostasis processes, there are limited studies to
															comprehensively investigate associations between sleep health and hemostasis
															in large population studies.
															Aims: We aim to investigate associations between sleep health and hemostatic
															factors in two large population-based cohort studies, the UK Biobank
															(UKBB) and the Netherlands Epidemiology of Obesity (NEO) study.
															Methods: Self-reported sleep duration derived from questionnaires was
															considered as exposure. As outcomes, we used seven hemostatic factors
															associated with the risk of venous thrombosis (factor [F]IX, FXI, fibrinogen,
															Von Willebrand factor, protein C, protein S, and antithrombin), measured
															using immunoassay-based OLINK Explore 3072. We performed linear
															regression analyses adjusting for sex, age, Townsend deprivation index,
															smoking, alcohol consumption, and body mass index. We repeated the
															analysis after excluding individuals diagnosed with a sleep disorder
															(n=2468) or reported using warfarin (n=744). To correct for multiple
															testing, we considered Bonferroni’s corrected p-value as a significant
															threshold (p < 0.007). Results: In 53,021 UKBB participants (mean age (SD): 56.8 (8.21),
																46% men), the mean sleep duration was 7.1 hours (SD: 1.13). Of the seven hemostatic
																factors, sleep duration was only associated with units decrease of Normalized
																Protein eXpression (NPX) of protein C (Beta: -0.004; p-value: 4.6×10-6) and
																antithrombin (Beta: -0.003; p-value: 8.2×10-8) per one-hour increase in sleep
																duration after multiple test correction. Results were consistent after adjustment
																for confounders and excluding additional participants. Results from the NEO study as
																well as results on other sleep health parameters will be available at the
																conference. Conclusion(s): The current analyses showed that longer habitual
																selfreported sleep duration is associated with lower anticoagulant factors while
																sleep duration was not associated with procoagulant factors. Further investigation
																is warranted to study the possible non-linear associations between sleep duration
																and hemostatic factors. <br><br>
																<b>Keywords:</b> biomarkers; sleep; VTE; UK Biobank
														</p>
													</div>
												</div>
											</div>
											<div class="modal-footer">
												<!-- <a href="" target="_blank">
																				<button type="button" class="btn btn-sm btn-default" >PubMed</button>
																				<a href="PodcastPage.html?" target="_blank">
																					<button type="button" class="btn btn-sm btn-default" >AI Podcast</button> -->
												<button type="button" class="btn btn-sm btn-default" data-dismiss="modal">Close</button>
											</div>
										</div>
									</div>
								</div>
								<!-- Modal end -->
							</div>

							<!--SHS-->
							
							<div class="col-sm-6 col-md-3 isotope-item Genomics Sleep MR AudioSum" style="top: 300px">
								<div class="image-box">
									<div class="overlay-container">
										<img src="images/SHS.jpg" alt="">
										<a class="overlay" data-toggle="modal" data-target="#project-8">
											<i class="fa fa-search-plus"></i> 
										</a>
									</div>
									<a class="btn btn-default btn-block" data-toggle="modal" data-target="#project-8">Published</a>
								</div>
								<h6 class="text-center" >Genome-wide association analysis of composite sleep health scores in 413,904 individuals </h6>

								<!-- Modal -->
								<div class="modal fade" id="project-8" tabindex="-1" role="dialog" aria-labelledby="project-8-label" aria-hidden="true">
									<div class="modal-dialog modal-lg">
										<div class="modal-content">
											<div class="modal-header">
												<button type="button" class="close" data-dismiss="modal"><span aria-hidden="true">&times;</span><span class="sr-only">Close</span></button>
												<h4 class="modal-title" id="project-8-label">Genome-wide association analysis of composite sleep health scores in 413,904 individuals </h4>
											</div>
											<div class="modal-body">
												
												<div class="row">												 
													<div class="col-md-12">
														<h7><i>Goodman MO, Faquih T, Paz V, Nagarajan P, Lane JM, Spitzer B, Maher M, Chung J, Cade BE, Purcell SM, Zhu X, Noordam R, Phillips AJK, Kyle SD, Spiegelhalder K, Weedon MN, Lawlor DA, Rotter JI, Taylor KD, Isasi CR, Sofer T, Dashti HS, Rutter MK, Redline S, Saxena R, Wang H. Genome-wide association analysis of composite sleep health scores in 413,904 individuals. Commun Biol. 2025 Jan 24;8(1):115. doi: 10.1038/s42003-025-07514-0. PMID: 39856408; PMCID: PMC11760956.</i>
														</h7><br>
														<p><b>Abstract</b><br>
															Recent genome-wide association studies (GWASs) of several individual sleep traits have identified hundreds of genetic loci, suggesting diverse mechanisms. Moreover, sleep traits are moderately correlated, so together may provide a more complete picture of sleep health, while illuminating distinct domains. Here we construct novel sleep health scores (SHSs) incorporating five core self-report measures: sleep duration, insomnia symptoms, chronotype, snoring, and daytime sleepiness, using additive (SHS-ADD) and five principal components-based (SHS-PCs) approaches. GWASs of these six SHSs identify 28 significant novel loci adjusting for multiple testing on six traits (p < 8.3e-9), along with 341 previously reported loci (p < 5e-08). The heritability of the first three SHS-PCs equals or exceeds that of SHS-ADD (SNP-h2 = 0.094), while revealing sleep-domain-specific genetic discoveries. Significant loci enrich in multiple brain tissues and in metabolic and neuronal pathways. Post-GWAS analyses uncover novel genetic mechanisms underlying sleep health and reveal connections (including potential causal links) to behavioral, psychological, and cardiometabolic traits.
															


															<br><br>
															<b>Keywords:</b> GWAS; sleep traits; PheWAS; chronotype; UK Biobank
														 </p>
													</div>
												</div>
											</div>
											<div class="modal-footer">
												 <a href="https://pubmed.ncbi.nlm.nih.gov/39856408/" target="_blank">
													<button type="button" class="btn btn-sm btn-default" >PubMed</button>
													<a href="PodcastPage.html?Pod=SHSpodcast" target="_blank">
														<button type="button" class="btn btn-sm btn-default" >AI Podcast</button>
												<button type="button" class="btn btn-sm btn-default" data-dismiss="modal">Close</button>
											</div>
										</div>
									</div>
								</div>
								<!-- Modal end -->
							</div>

							<!--EDS1-->
							
							<div class="col-sm-6 col-md-3 isotope-item Metabolomics Genomics Sleep Etiology MR" style="top: 300px">
								<div class="image-box">
									<div class="overlay-container">
										<img src="images/EDSmetabolomics.png" alt="">
										<a class="overlay" data-toggle="modal" data-target="#project-9">
											<i class="fa fa-search-plus"></i> 
										</a>
									</div>
									<a class="btn btn-default btn-block" data-toggle="modal" data-target="#project-9">Preprint</a>
								</div>
								<h6 class="text-center">Steroid Hormone Biosynthesis and Dietary Related Metabolites Associated with Excessive Daytime Sleepiness</h6>

								<!-- Modal -->
								<div class="modal fade" id="project-9" tabindex="-1" role="dialog" aria-labelledby="project-8-label" aria-hidden="true">
									<div class="modal-dialog modal-lg">
										<div class="modal-content">
											<div class="modal-header">
												<button type="button" class="close" data-dismiss="modal"><span aria-hidden="true">&times;</span><span class="sr-only">Close</span></button>
												<h4 class="modal-title" id="project-9-label">Steroid Hormone Biosynthesis and Dietary Related Metabolites Associated with Excessive Daytime Sleepiness</h4>
											</div>
											<div class="modal-body">
												
												<div class="row">												 
													<div class="col-md-12">
														<h7><i>Faquih T, Potts K, Yu B, Kaplan R, Isasi CR, Qi Q, Taylor KD, Liu PY, Tracy RP, Johnson C, Rich SS, Clish CB, Gerzsten RE, Rotter JI, Redline S, Sofer T, Wang H. Steroid Hormone Biosynthesis and Dietary Related Metabolites Associated with Excessive Daytime Sleepiness. medRxiv [Preprint]. 2024 Sep 13:2024.09.12.24313561. doi: 10.1101/2024.09.12.24313561. PMID: 39314973; PMCID: PMC11419218. </i>
														</h7><br>
														<p><b>Abstract</b><br>
															Background: Excessive daytime sleepiness (EDS) is a complex sleep problem that affects approximately 33% of the United
															States population. Although EDS usually occurs in conjunction with insufficient sleep, and other sleep and circadian
															disorders, recent studies have shown unique genetic markers and metabolic pathways underlying EDS. Here, we aimed to
															further elucidate the biological profile of EDS using large scale single- and pathway-level metabolomics analyses.
															
															Methods: Metabolomics data were available for 877 metabolites in 6,071 individuals from the Hispanic Community Health
															Study/Study of Latinos (HCHS/SOL) and EDS was assessed using the Epworth Sleepiness Scale (ESS) questionnaire. We
															performed linear regression for each metabolite on continuous ESS, adjusting for demographic, lifestyle, and
															physiological confounders, and in sex specific groups. Subsequently, gaussian graphical modelling was performed coupled
															with pathway and enrichment analyses to generate a holistic interactive network of the metabolomic profile of EDS
															associations.
															
															Findings: We identified seven metabolites belonging to steroids, sphingomyelin, and long chain fatty acids sub-pathways
															in the primary model associated with EDS, and an additional three metabolites in the male-specific analysis. The
															identified metabolites particularly played a role in steroid hormone biosynthesis.
															
															Interpretation: Our findings indicate that an EDS metabolomic profile is characterized by endogenous and dietary
															metabolites within the steroid hormone biosynthesis pathway, with some pathways that differ by sex. Our findings
															identify potential pathways to target for addressing the causes or consequences of EDS and related sleep disorders.
															<br><br>
															<b>Keywords: </b>metabolomics, excessive daytime sleepiness, sex steroid hormones, nutrition, network analysis, Mendelian randomization
														 </p>
													</div>
												</div>
											</div>
											<div class="modal-footer">
												<a href="https://papers.ssrn.com/sol3/papers.cfm?abstract_id=4955294" target="_blank">
													<button type="button" class="btn btn-sm btn-default" >Preprint in The Lancet (SSRN)</button> </a>
													<a href="https://tofaquih.github.io/Metabolomics-Excessive-Daytime-Sleepiness/" target="_blank">
														<button type="button" class="btn btn-sm btn-default" >Enrichment GMM Networks</button> </a>
													
												<button type="button" class="btn btn-sm btn-default" data-dismiss="modal">Close</button>
											</div>
										</div>
									</div>
								</div>
								<!-- Modal end -->
							</div>


							<!--gene_eds-osa-->
							
							<div class="col-sm-6 col-md-3 isotope-item Genomics Sleep Etiology" style="top: 300px">
								<div class="image-box">
									<div class="overlay-container">
										<img src="images/edsosa.png" alt="">
										<a class="overlay" data-toggle="modal" data-target="#project-13">
											<i class="fa fa-search-plus"></i> 
										</a>
									</div>
									<a class="btn btn-default btn-block" data-toggle="modal" data-target="#project-13">Preprint</a>
								</div>
								<h6 class="text-center">Gene-Excessive Sleepiness Interactions Suggest Treatment Targets for Obstructive Sleep Apnea Subtype</h6>

								<!-- Modal -->
								<div class="modal fade" id="project-13" tabindex="-1" role="dialog" aria-labelledby="project-13-label" aria-hidden="true">
									<div class="modal-dialog modal-lg">
										<div class="modal-content">
											<div class="modal-header">
												<button type="button" class="close" data-dismiss="modal"><span aria-hidden="true">&times;</span><span class="sr-only">Close</span></button>
												<h4 class="modal-title" id="project-7-label">Gene-Excessive Sleepiness Interactions Suggest Treatment Targets for Obstructive Sleep Apnea Subtype </h4>
											</div>
											<div class="modal-body">
												
												<div class="row">												 
													<div class="col-md-12">
														<h7><i>Nagarajan P, Kurniansyah N, Lee J, Gharib SA, Xu Y, Zhang Y, Spitzer B, Faquih T, Zhou H, Boerwinkle E, Chen H, Gottlieb DJ, Guo X, Heard-Costa NL, Hidalgo BA, Levy D, Liu PY, Mei H, Montalvan R, Mukherjee S, North KE, O'Conner GT, Palmer LJ, Patel SR, Psaty BM, Purcell SM, Raffield LM, Rich SS, Rotter JI, Saxena R, Smith AV, Stone KL, Zhu X; TOPMed Sleep Trait WG; Cade BE, Sofer T, Redline S, Wang H. Gene-Excessive Sleepiness Interactions Suggest Treatment Targets for Obstructive Sleep Apnea Subtype. medRxiv [Preprint]. 2024 Oct 28:2024.10.25.24316158. doi: 10.1101/2024.10.25.24316158. PMID: 39574859; PMCID: PMC11581070.</i>
														</h7><br>
														<p><b>Abstract</b>

															Obstructive sleep apnea (OSA) is a multifactorial sleep disorder characterized by a strong genetic basis. Excessive daytime sleepiness (EDS) is a symptom that is reported by a subset of OSA patients, persisting even after treatment with continuous positive airway pressure (CPAP). It is recognized as a clinical subtype underlying OSA carrying alarming heightened cardiovascular risk. Thus, conceptualizing EDS as an exposure variable, we sought to investigate EDS's influence on genetic variation linked to apnea-hypopnea index (AHI), a diagnostic measure of OSA severity. This study serves as the first large-scale genome-wide gene x environment interaction analysis for AHI, investigating the interplay between its genetic markers and EDS across and within specific sex. Our work pools together whole genome sequencing data from seven cohorts, enabling a diverse dataset (four population backgrounds) of over 11,500 samples. Among the total 16 discovered genetic targets with interaction evidence with EDS, eight are previously unreported for OSA, including CCDC3, MARCHF1, and MED31 identified in all sexes; TMEM26, CPSF4L, and PI4K2B identified in males; and RAP1GAP and YY1 identified in females. We discuss connections to insulin resistance, thiamine deficiency, and resveratrol use that may be worthy of therapeutic consideration for excessively sleepy OSA patients.
															<br>
													

															<br><br>
															<b>Keywords:</b>OSA; EDS; Genetics</p>
													</div>
												</div>
											</div>
											<div class="modal-footer">
												<a href="https://pubmed.ncbi.nlm.nih.gov/39574859/" target="_blank">
													<button type="button" class="btn btn-sm btn-default" >Preprint</button> </a>

												<button type="button" class="btn btn-sm btn-default" data-dismiss="modal">Close</button>
											</div>
										</div>
									</div>
								</div>
								<!-- Modal end -->
							</div>

							
							

							<!--MAGE-->
							
							<div class="col-sm-6 col-md-3 isotope-item Prediction Metabolomics AudioSum" style="top: 300px">
								<div class="image-box">
									<div class="overlay-container">
										<img src="images/portfolio-7.png" alt="">
										<a class="overlay" data-toggle="modal" data-target="#project-7">
											<i class="fa fa-search-plus"></i> 
										</a>
									</div>
									<a class="btn btn-default btn-block" data-toggle="modal" data-target="#project-7">Published</a>
								</div>
								<h6 class="text-center">Robust Metabolomic Age Prediction Based on a Wide Selection of Metabolites</h6>

								<!-- Modal -->
								<div class="modal fade" id="project-7" tabindex="-1" role="dialog" aria-labelledby="project-7-label" aria-hidden="true">
									<div class="modal-dialog modal-lg">
										<div class="modal-content">
											<div class="modal-header">
												<button type="button" class="close" data-dismiss="modal"><span aria-hidden="true">&times;</span><span class="sr-only">Close</span></button>
												<h4 class="modal-title" id="project-7-label">Robust Metabolomic Age Prediction Based on a Wide Selection of Metabolites </h4>
											</div>
											<div class="modal-body">
												
												<div class="row">												 
													<div class="col-md-12">
														<h7><i>Faquih TO, van Hylckama Vlieg A, Surendran P, Butterworth AS, Li-Gao R, de Mutsert R, Rosendaal FR, Noordam R, van Heemst D, Willems van Dijk K, Mook-Kanamori DO. Robust Metabolomic Age Prediction Based on a Wide Selection of Metabolites. J Gerontol A Biol Sci Med Sci. 2025 Feb 10;80(3):glae280. doi: 10.1093/gerona/glae280. PMID: 39821408; PMCID: PMC11809259.</i>
														</h7><br>
														<p><b>Abstract</b><br>
															

Chronological age is a major risk factor for numerous diseases. However, chronological age does not capture the complex
biological aging process. The difference between chronological age and biologically driven aging could be more
informative in reflecting health status. Here, we set out to develop a metabolomic age prediction model by applying
ridge regression and bootstrapping with 826 metabolites (678 endogenous and 148 xenobiotics) measured by an untargeted
platform in relatively healthy blood donors aged 18-75 years from the INTERVAL study (N = 11 977; 50.2% men). After
bootstrapping internal validation, the metabolomic age prediction models demonstrated high performance with an adjusted
R2 of 0.83 using all metabolites and 0.82 using only endogenous metabolites. The former was significantly associated
with obesity and cardiovascular disease in the Netherlands Epidemiology of Obesity study (N = 599; 47.0% men; age range
= 45-65) due to the contribution of medication-derived metabolites-namely salicylate and ibuprofen-and environmental
exposures such as cotinine. Additional metabolomic age prediction models using all metabolites were developed for men
and women separately. The models had high performance (R² = 0.85 and 0.86) but shared a moderate correlation of 0.72.
Furthermore, we observed 163 sex-dimorphic metabolites, including threonine, glycine, cholesterol, and androgenic and
progesterone-related metabolites. Our strongest predictors across all models were novel and included hydroxyasparagine
(Model Endo + Xeno β = 4.74), vanillylmandelate (β = 4.07), and 5,6-dihydrouridine (β = -4.2). Our study presents a
robust metabolomic age model that reveals distinct sex-based age-related metabolic patterns and illustrates the value of
including xenobiotic to enhance metabolomic prediction accuracy.

															<br><br>
															<b>Keywords:</b>
														 Aging, metabolomic age, untargeted metabolomics, prediction modelling</p>
													</div>
												</div>
											</div>
											<div class="modal-footer">
												<a href="https://pubmed.ncbi.nlm.nih.gov/39821408/" target="_blank">
													<button type="button" class="btn btn-sm btn-default" >Published</button> </a>
													<a href="PodcastPage.html?Pod=MetaboAgePod" target="_blank">
														<button type="button" class="btn btn-sm btn-default" >AI Podcast</button>

												<button type="button" class="btn btn-sm btn-default" data-dismiss="modal">Close</button>
											</div>
										</div>
									</div>
								</div>
								<!-- Modal end -->
							</div>

							<!--missingnessGWAS-->
							
							<div class="col-sm-6 col-md-3 isotope-item Etiology Metabolomics Genomics" style="top: 300px">
								<div class="image-box">
									<div class="overlay-container">
										<img src="images/MMGWAS.png" alt="">
										<a class="overlay" data-toggle="modal" data-target="#project-14">
											<i class="fa fa-search-plus"></i> 
										</a>
									</div>
									<a class="btn btn-default btn-block" data-toggle="modal" data-target="#project-14">Preprint</a>
								</div>
								<h6 class="text-center">Genome-wide Association Studies of Missing Metabolite Measures: Results From Two Population-based Studies</h6>

								<!-- Modal -->
								<div class="modal fade" id="project-14" tabindex="-1" role="dialog" aria-labelledby="project-14-label" aria-hidden="true">
									<div class="modal-dialog modal-lg">
										<div class="modal-content">
											<div class="modal-header">
												<button type="button" class="close" data-dismiss="modal"><span aria-hidden="true">&times;</span><span class="sr-only">Close</span></button>
												<h4 class="modal-title" id="project-14-label">Genome-wide Association Studies of Missing Metabolite Measures: Results From Two Population-based Studies</h4>
											</div>
											<div class="modal-body">
												
												<div class="row">												 
													<div class="col-md-12">
														<h7><i> Genome-wide Association Studies of Missing Metabolite Measures: Results From Two Population-based Studies
															Tariq O. Faquih, Mohammed Aslam Imtiaz, Valentina Talevi, Elvire N. Landstra, Astrid van Hylckama Vlieg, Ruifang Li-Gao, Frits R. Rosendaal, Raymond Noordam, Diana van Heemst, Dennis O. Mook-Kanamori, Monique M. B. Breteler, N. Ahmad Aziz, Ko Willems van Dijk
															medRxiv 2024.10.02.24314800; doi: https://doi.org/10.1101/2024.10.02.24314800 </i>
														</h7><br>
														<p><b>Abstract</b><br>
															Metabolomic studies are increasingly used for both etiological and predictive research, but frequently report missing values. We hypothesized that interindividual genetic variation may account for part of this missingness. Therefore, we performed a GWAS of missingness in measured metabolite levels using an untargeted mass spectrometry-based platform in the Netherlands Epidemiology of Obesity Study (N=594) and the Rhineland Study (N=4,165). We considered metabolites missing in 10%-90% of individuals in both cohorts (N=224). GWAS meta-analyses of these metabolites’ probability of missingness revealed 55 metabolome-wide significant associations, including 42 novel ones (p<1.58E-10), involving 28 metabolites and 41 lead SNPs. Despite considerable pleiotropy, the majority of identified SNP-‘missing metabolite’ associations were biologically plausible, relating to beta-oxidation, bile acids, steroids, and xenobiotics metabolism. These findings suggest that missing values in metabolomics are nonrandom and partly reflect genetic variation, accounting for which is important for both clinical and epidemiological studies, especially nutritional and pharmacogenetics studies.

															<br><br>
															<b>Keywords:</b>
														 Untargeted metabolomics, GWAS</p>
													</div>
												</div>
											</div>
											<div class="modal-footer">
												<a href="https://www.medrxiv.org/content/10.1101/2024.10.02.24314800v3" target="_blank">
													<button type="button" class="btn btn-sm btn-default" >Preprint</button> </a>

													<a href="https://www.ashg.org/wp-content/uploads/2023/10/ASHG2023-PlatformAbstracts.pdf" target="_blank">
														<button type="button" class="btn btn-sm btn-default" >ASHG2023 Platform Abstracts (PDF)</button> </a>
													
													<a href="https://tofaquih.github.io/GWASMissingMetabolites/" target="_blank">
														<button type="button" class="btn btn-sm btn-default" >GGM Network</button> </a>

												

												<button type="button" class="btn btn-sm btn-default" data-dismiss="modal">Close</button>
											</div>
										</div>
									</div>
								</div>
								<!-- Modal end -->
							</div>
							
							
							<!--lungfunction-->
														
							<div class="col-sm-6 col-md-3 isotope-item Genomics Prediction Etiology" style="top: 300px">
								<div class="image-box">
									<div class="overlay-container">
										<img src="images/lung function.jpg" alt="">
										<a class="overlay" data-toggle="modal" data-target="#project-10">
											<i class="fa fa-search-plus"></i> 
										</a>
									</div>
									<a class="btn btn-default btn-block" data-toggle="modal" data-target="#project-10">Published</a>
								</div>
								<h6 class="text-center">Multi-ancestry GWAS improve resolution of genes and pathways influencing lung function and COPD risk
									<br></h6>

								<!-- Modal -->
								<div class="modal fade" id="project-10" tabindex="-1" role="dialog" aria-labelledby="project-8-label" aria-hidden="true">
									<div class="modal-dialog modal-lg">
										<div class="modal-content">
											<div class="modal-header">
												<button type="button" class="close" data-dismiss="modal"><span aria-hidden="true">&times;</span><span class="sr-only">Close</span></button>
												<h4 class="modal-title" id="project-10-label">Multi-ancestry genome-wide association analyses improve resolution of genes and pathways influencing lung function and chronic obstructive pulmonary disease risk
												</h4>
											</div>
											<div class="modal-body">
												
												<div class="row">												 
													<div class="col-md-12">
														<h7><i> Shrine N, Izquierdo AG, et al. Multi-ancestry genome-wide association analyses improve resolution of genes and pathways influencing lung function and chronic obstructive pulmonary disease risk. Nat Genet. 2023 Mar;55(3):410-422. doi: 10.1038/s41588-023-01314-0. Epub 2023 Mar 13. Erratum in: Nat Genet. 2023 Oct;55(10):1778-1779. PMID: 36914875; PMCID: PMC10011137.</i>
														</h7><br>
														<p><b>Abstract</b><br>Lung-function impairment underlies chronic obstructive pulmonary disease (COPD) and predicts mortality. In the largest multi-ancestry genome-wide association meta-analysis of lung function to date, comprising 588,452 participants, we identified 1,020 independent association signals implicating 559 genes supported by ≥2 criteria from a systematic variant-to-gene mapping framework. These genes were enriched in 29 pathways. Individual variants showed heterogeneity across ancestries, age and smoking groups, and collectively as a genetic risk score showed strong association with COPD across ancestry groups. We undertook phenome-wide association studies for selected associated variants as well as trait and pathway-specific genetic risk scores to infer possible consequences of intervening in pathways underlying lung function. We highlight new putative causal variants, genes, proteins and pathways, including those targeted by existing drugs. These findings bring us closer to understanding the mechanisms underlying lung function and COPD, and should inform functional genomics experiments and potentially future COPD therapies.


															<br><br>
															<b>Keywords:</b> COPD; GWAS, PheWAS; PRS; lung function
														</p>
													</div>
												</div>
											</div>
											<div class="modal-footer">
												<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10011137/" target="_blank">
													<button type="button" class="btn btn-sm btn-default" >Full text</button> </a>

												<button type="button" class="btn btn-sm btn-default" data-dismiss="modal">Close</button>
											</div>
										</div>
									</div>
								</div>
								<!-- Modal end -->
							</div>


							<!--t2d-->
														
							<div class="col-sm-6 col-md-3 isotope-item Genomics" style="top: 300px">
								<div class="image-box">
									<div class="overlay-container">
										<img src="images/t2d2022.jpg" alt="">
										<a class="overlay" data-toggle="modal" data-target="#project-12">
											<i class="fa fa-search-plus"></i> 
										</a>
									</div>
									<a class="btn btn-default btn-block" data-toggle="modal" data-target="#project-12">Published</a>
								</div>
								<h6 class="text-center">Type 2 Diabetes Partitioned Polygenic Scores Associate With Disease Outcomes in 454,193 Individuals Across 13 Cohorts
									<br></h6>

								<!-- Modal -->
								<div class="modal fade" id="project-12" tabindex="-1" role="dialog" aria-labelledby="project-8-label" aria-hidden="true">
									<div class="modal-dialog modal-lg">
										<div class="modal-content">
											<div class="modal-header">
												<button type="button" class="close" data-dismiss="modal"><span aria-hidden="true">&times;</span><span class="sr-only">Close</span></button>
												<h4 class="modal-title" id="project-12-label">Type 2 Diabetes Partitioned Polygenic Scores Associate With Disease Outcomes in 454,193 Individuals Across 13 Cohorts

												</h4>
											</div>
											<div class="modal-body">
												
												<div class="row">												 
													<div class="col-md-12">
														<h7><i> DiCorpo D, et al. Type 2 Diabetes Partitioned Polygenic Scores Associate With Disease Outcomes in 454,193 Individuals Across 13 Cohorts. Diabetes Care. 2022 Mar 1;45(3):674-683. doi: 10.2337/dc21-1395. PMID: 35085396; PMCID: PMC8918228.</i></h7><br>
														<p><b>Abstract</b><br>Objective: Type 2 diabetes (T2D) has heterogeneous patient clinical characteristics and outcomes. In previous work, we investigated the genetic basis of this heterogeneity by clustering 94 T2D genetic loci using their associations with 47 diabetes-related traits and identified five clusters, termed β-cell, proinsulin, obesity, lipodystrophy, and liver/lipid. The relationship between these clusters and individual-level metabolic disease outcomes has not been assessed.

															Research design and methods: Here we constructed individual-level partitioned polygenic scores (pPS) for these five clusters in 12 studies from the Cohorts for Heart and Aging Research in Genomic Epidemiology (CHARGE) consortium and the UK Biobank (n = 454,193) and tested for cross-sectional association with T2D-related outcomes, including blood pressure, renal function, insulin use, age at T2D diagnosis, and coronary artery disease (CAD).
															
															Results: Despite all clusters containing T2D risk-increasing alleles, they had differential associations with metabolic outcomes. Increased obesity and lipodystrophy cluster pPS, which had opposite directions of association with measures of adiposity, were both significantly associated with increased blood pressure and hypertension. The lipodystrophy and liver/lipid cluster pPS were each associated with CAD, with increasing and decreasing effects, respectively. An increased liver/lipid cluster pPS was also significantly associated with reduced renal function. The liver/lipid cluster includes known loci linked to liver lipid metabolism (e.g., GCKR, PNPLA3, and TM6SF2), and these findings suggest that cardiovascular disease risk and renal function may be impacted by these loci through their shared disease pathway.
															
															Conclusions: Our findings support that genetically driven pathways leading to T2D also predispose differentially to clinical outcomes.


															<br><br>
															<b>Keywords:</b> Type 2 Diabetes; GWAS; PRS; Obesity
														</p>
													</div>
												</div>
											</div>
											<div class="modal-footer">
												<a href="https://diabetesjournals.org/care/article/45/3/674/141024/Type-2-Diabetes-Partitioned-Polygenic-Scores" target="_blank">
													<button type="button" class="btn btn-sm btn-default" >Full text</button> </a>

												<button type="button" class="btn btn-sm btn-default" data-dismiss="modal">Close</button>
											</div>
										</div>
									</div>
								</div>
								<!-- Modal end -->
							</div>




							<div class="col-sm-6 col-md-3 isotope-item Metabolomics Methodology">
								<div class="image-box">
									<div class="overlay-container">
										<img src="images/portfolio-1.png" alt="">
										<a class="overlay" data-toggle="modal" data-target="#project-1">
											<i class="fa fa-search-plus"></i>
											<a class="btn btn-default btn-block" data-toggle="modal" data-target="#project-1">Published</a>
										</a>
									</div>
									
									<h6 class="text-center">A Workflow for Missing Values Imputation of Untargeted Metabolomics Data<br><br></h6> 
								</div>
								<!-- Modal -->
								<div class="modal fade" id="project-1" tabindex="-1" role="dialog" aria-labelledby="project-1-label" aria-hidden="true">
									<div class="modal-dialog modal-lg">
										<div class="modal-content">
											<div class="modal-header">
												<button type="button" class="close" data-dismiss="modal"><span aria-hidden="true">&times;</span><span class="sr-only">Close</span></button>
												<h4 class="modal-title" id="project-1-label">
													A Workflow for Missing Values Imputation of Untargeted Metabolomics Data (2020)</h4>
											</div>
											<div class="modal-body">
												
												<div class="row">												 
													<div class="col-md-12">
														<h7><i>Faquih T, van Smeden M, Luo J, le Cessie S, Kastenmüller G, Krumsiek J, Noordam R, van Heemst D, Rosendaal FR, van Hylckama Vlieg A, Willems van Dijk K, Mook-Kanamori DO. A Workflow for Missing Values Imputation of Untargeted Metabolomics Data. Metabolites. 2020 Nov 26;10(12):486. doi: 10.3390/metabo10120486. PMID: 33256233; PMCID: PMC7761057.</i>
														</h7><br>
														<p><b>Abstract</b><br>
															Metabolomics studies have seen a steady growth due to the development and implementation of affordable and high-quality metabolomics platforms. In large metabolite panels, measurement values are frequently missing and, if neglected or sub-optimally imputed, can cause biased study results. We provided a publicly available, user-friendly R script to streamline the imputation of missing endogenous, unannotated, and xenobiotic metabolites. We evaluated the multivariate imputation by chained equations (MICE) and k-nearest neighbors (kNN) analyses implemented in our script by simulations using measured metabolites data from the Netherlands Epidemiology of Obesity (NEO) study (n = 599). We simulated missing values in four unique metabolites from different pathways with different correlation structures in three sample sizes (599, 150, 50) with three missing percentages (15%, 30%, 60%), and using two missing mechanisms (completely at random and not at random). Based on the simulations, we found that for MICE, larger sample size was the primary factor decreasing bias and error. For kNN, the primary factor reducing bias and error was the metabolite correlation with its predictor metabolites. MICE provided consistently higher performance measures particularly for larger datasets (n > 50). In conclusion, we presented an imputation workflow in a publicly available R script to impute untargeted metabolomics data. Our simulations provided insight into the effects of sample size, percentage missing, and correlation structure on the accuracy of the two imputation methods.
															<br><br>
															<b>Keywords:</b> imputation; k-nearest neighbors; metabolon; multiple imputation using chained equations; simulation; untargeted metabolomics; workflow. 
															
														</p>
													</div>
												</div>
											</div>
											<div class="modal-footer">
												<a href="https://doi.org/10.3390/metabo10120486" target="_blank">
													<button type="button" class="btn btn-sm btn-default" >Full Text</button> </a>

												<a href="https://pubmed.ncbi.nlm.nih.gov/33256233/" target="_blank">
												<button type="button" class="btn btn-sm btn-default" >PubMed</button> </a>

												<a href="https://github.com/tofaquih/imputation_of_untargeted_metabolites/tree/v1.3" target="_blank">
													<button type="button" class="btn btn-sm btn-default" >GitHub Repo</button> </a>

												<button type="button" class="btn btn-sm btn-default" data-dismiss="modal">Close</button>
											</div>
										</div>
									</div>
								</div>
								<!-- Modal end -->
							</div>

		
							<!--PFAS-->
							
							<div class="col-sm-6 col-md-3 isotope-item Etiology Metabolomics Etiology AudioSum" style="top: 300px">
								<div class="image-box">
									<div class="overlay-container">
										<img src="images/portfolio-5.png" alt="">
										<a class="overlay" data-toggle="modal" data-target="#project-5">
											<i class="fa fa-search-plus"></i> 
										</a>
									</div>
									<a class="btn btn-default btn-block" data-toggle="modal" data-target="#project-5">Published</a>
								</div>
								<h6 class="text-center">PFAS concentrations are associated with an unfavorable cardio-metabolic risk profile<br><br></h6>

								<!-- Modal -->
								<div class="modal fade" id="project-5" tabindex="-1" role="dialog" aria-labelledby="project-5-label" aria-hidden="true">
									<div class="modal-dialog modal-lg">
										<div class="modal-content">
											<div class="modal-header">
												<button type="button" class="close" data-dismiss="modal"><span aria-hidden="true">&times;</span><span class="sr-only">Close</span></button>
												<h4 class="modal-title" id="project-5-label">Per- and polyfluoroalkyl substances concentrations are associated with an unfavorable cardio-metabolic risk profile: findings from two population-based cohort studies</h4>
											</div>
											<div class="modal-body">
												
												<div class="row">												 
													<div class="col-md-12">
														<h7><i>Faquih, T.O., Landstra, E.N., van Hylckama Vlieg, A. et al. Per- and Polyfluoroalkyl Substances Concentrations are Associated with an Unfavorable Cardio-Metabolic Risk Profile: Findings from Two Population-Based Cohort Studies. Expo Health (2024). https://doi.org/10.1007/s12403-023-00622-4
														</i>
														</h7><br>
														<p><b>Layman Summary: </b>Our new study, jointly conducted by researchers from Leiden University (LUMC) and the German Center of Neurodegenerative Diseases (DZNE) reports the harmful effects of PFAS, aka "Forever Chemicals", on the people living in the cities of Leiden and Bonn.
															<br>
															⚠️We found that, surprisingly, 99.9% of all individuals in the study had detectable levels of three PFAS chemicals.
															<br>
															📈Those chemicals were linked to harmful levels of cholesterols known to increase the risk of cardiovascular disease, especially in younger individuals.
															<br>
															🚫The evidence shows that even PFAS levels previously considered "safe" are, in fact, detrimental to the overall health of the population.
															</p>
														<p><b>Abstract</b><br>
															Per- and polyfluoroalkyl substances (PFAS) are widely used and persistent chemicals, leading to ubiquitous exposure. Although high PFAS levels have been associated with an adverse cardiovascular risk profile, the distribution of levels and relations with cardio-metabolic risk markers in the general population have not been fully characterized. We assessed the association between blood levels of perfluorooctaneic acid (PFOA), perfluorooctane sulfonic acid (PFOS), and perfluorohexanesulfonic acid (PFHxS) and a range of lipoproteins and metabolites as well as clinical lipid measurements. We used data from participants of the Netherlands Epidemiology of Obesity study (NEO) (n= 584) and the Rhineland Study (n= 1,962), jointly spanning an age range of 30 to 89 years. PFAS were measured with the Metabolon HD4 platform, and lipoprotein and metabolite profiles were measured using Nightingale's nuclear magnetic resonance-spectroscopy platform, and mainly comprised lipoprotein markers. Using linear regression analyses, we quantified age-, sex- and education-adjusted associations of PFOA, PFOS, and PFHxS with clinical lipid measurements and 224 lipoproteins and metabolites. Higher levels of PFAS, particularly PFOS and PFHxS, were associated with higher concentrations of total lipid, cholesterol and phospholipid content in most HDL, IDL, LDL and VLDL subclasses. The effect sizes were age-dependent for the majority of the associations, with the deleterious effects of PFAS being generally stronger in people below compared to those above median age. Our observation that in the general population even low PFAS concentrations are associated with an unfavorable lipid profile, calls for further critical regulation of PFAS substances.
															<br><br>
															<b>Keywords:</b> polyfluoroalkyl substances, environment, metabolomics, lipoproteins, cardiovascular </p>
													</div>
												</div>
											</div>
											<div class="modal-footer">
												<a href="https://link.springer.com/article/10.1007/s12403-023-00622-4" target="_blank">
													<button type="button" class="btn btn-sm btn-default" >Full text</button> </a>

													<a href="PodcastPage.html?Pod=pfaspod" target="_blank">
														<button type="button" class="btn btn-sm btn-default" >AI Podcast</button>

													<button type="button" class="btn btn-sm btn-default" >Preprint</button> </a>

													

												<button type="button" class="btn btn-sm btn-default" data-dismiss="modal">Close</button>
											</div>
										</div>
									</div>
								</div>
								<!-- Modal end -->
							</div>


							
							<div class="col-sm-6 col-md-3 isotope-item Etiology Metabolomics">
								<div class="image-box">
									<div class="overlay-container">
										<img src="images/portfolio-3.png" alt="">
										<a class="overlay" data-toggle="modal" data-target="#project-3">
											<i class="fa fa-search-plus"></i> 
										</a>
									</div>
									<a class="btn btn-default btn-block" data-toggle="modal" data-target="#project-3">Published</a>
								</div>
								<h6 class="text-center">Hepatic triglyceride content is intricately associated with numerous metabolites and biochemical pathways</h6>

								<!-- Modal -->
								<div class="modal fade" id="project-3" tabindex="-1" role="dialog" aria-labelledby="project-3-label" aria-hidden="true">
									<div class="modal-dialog modal-lg">
										<div class="modal-content">
											<div class="modal-header">
												<button type="button" class="close" data-dismiss="modal"><span aria-hidden="true">&times;</span><span class="sr-only">Close</span></button>
												<h4 class="modal-title" id="project-3-label">Hepatic triglyceride content is intricately associated with numerous metabolites and biochemical pathways</h4>
											</div>
											<div class="modal-body">
												
												<div class="row">												 
													<div class="col-md-12">
														<h7><i>Faquih TO, van Klinken JB, Li-Gao R, Noordam R, van Heemst D, Boone S, Sheridan PA, Michelotti G, Lamb H, de Mutsert R, Rosendaal FR, van Hylckama Vlieg A, van Dijk KW, Mook-Kanamori DO. Hepatic triglyceride content is intricately associated with numerous metabolites and biochemical pathways. Liver Int. 2023 Jul;43(7):1458-1472. doi: 10.1111/liv.15575. Epub 2023 Apr 5. PMID: 37017544.</i>
														</h7><br>
														<p><b>Abstract</b><br>
															
												Background and Aims

												Non-alcoholic fatty liver disease (NAFLD) is characterized by the pathological accumulation of triglycerides in hepatocytes and is associated with insulin resistance, atherogenic dyslipidaemia and cardiometabolic diseases. Thus far, the extent of metabolic dysregulation associated with hepatic triglyceride accumulation has not been fully addressed. In this study, we aimed to identify metabolites associated with hepatic triglyceride content (HTGC) and map these associations using network analysis.
												Methods

												To gain insight in the spectrum of metabolites associated with hepatic triglyceride accumulation, we performed a comprehensive plasma metabolomics screening of 1363 metabolites in apparently healthy middle aged (age 45–65) individuals (N = 496) in whom HTGC was measured by proton magnetic resonance spectroscopy. An atlas of metabolite–HTGC associations, based on univariate results, was created using correlation-based Gaussian graphical model (GGM) and genome scale metabolic model network analyses. Pathways associated with the clinical prognosis marker fibrosis 4 (FIB-4) index were tested using a closed global test.
												Results

												Our analyses revealed that 118 metabolites were univariately associated with HTGC (p-value <6.59 × 10−5), including 106 endogenous, 1 xenobiotic and 11 partially characterized/uncharacterized metabolites. These associations were mapped to several biological pathways including branched amino acids (BCAA), diglycerols, sphingomyelin, glucosyl-ceramide and lactosyl-ceramide. We also identified a novel possible HTGC-related pathway connecting glutamate, metabolonic lactone sulphate and X-15245 using the GGM network. These pathways were confirmed to be associated with the FIB-4 index as well. The full interactive metabolite-HTGC atlas is provided online: https://tofaquih.github.io/AtlasLiver/.
												Conclusions

												The combined network and pathway analyses indicated extensive associations between BCAA and the lipids pathways with HTGC and the FIB-4 index. Moreover, we report a novel pathway glutamate-metabolonic lactone sulphate-X-15245 with a potential strong association with HTGC. These findings can aid elucidating HTGC metabolomic profiles and provide insight into novel drug targets for fibrosis-related outcomes.

															<br><br>
															<b>Keywords:</b> liver triglyceride content; metabolomics; pathway analysis </p>
													</div>
												</div>
											</div>
											<div class="modal-footer">

												<a href="https://tofaquih.github.io/AtlasLiver/" target="_blank">
												<button type="button" class="btn btn-sm btn-default" >Website and Networks</button> </a>

												<button type="button" class="btn btn-sm btn-default" data-dismiss="modal">Close</button>
											</div>
										</div>
									</div>
								</div>
								<!-- Modal end -->
							</div>

							<!--CAG-->
							<div class="col-sm-6 col-md-3 isotope-item Etiology Metabolomics Genomics Etiology">
								<div class="image-box">
									<div class="overlay-container">
										<img src="images/portfolio-6.png" alt="">
										<a class="overlay" data-toggle="modal" data-target="#project-6">
											<i class="fa fa-search-plus"></i> 
										</a>
									</div>
									<a class="btn btn-default btn-block" data-toggle="modal" data-target="#project-6">Published</a>
								</div>
								<h6 class="text-center">Normal range CAG repeat size variations in the HTT gene and their assoction with lipoproteins profile</h6>

								<!-- Modal -->
								<div class="modal fade" id="project-6" tabindex="-1" role="dialog" aria-labelledby="project-6-label" aria-hidden="true">
									<div class="modal-dialog modal-lg">
										<div class="modal-content">
											<div class="modal-header">
												<button type="button" class="close" data-dismiss="modal"><span aria-hidden="true">&times;</span><span class="sr-only">Close</span></button>
												<h4 class="modal-title" id="project-6-label">Normal range CAG repeat size variations in the HTT gene are associated with an adverse lipoprotein profile partially mediated by body mass index</h4>
											</div>
											<div class="modal-body">
												
												<div class="row">												 
													<div class="col-md-12">
														<h7><i>

															Tariq O Faquih, N Ahmad Aziz, Sarah L Gardiner, Ruifang Li-Gao, Renée de Mutsert, Yuri Milaneschi, Stella Trompet, J Wouter Jukema, Frits R Rosendaal, Astrid Hylckama Vlieg, Ko Willems Dijk, Dennis O Mook-Kanamori, Normal range CAG repeat size variations in the HTT gene are associated with an adverse lipoprotein profile partially mediated by body mass index, Human Molecular Genetics, 2023;, ddad020, https://doi.org/10.1093/hmg/ddad020 </i>
														</h7><br>
														<p><b>Abstract</b><br>
															Tandem CAG repeat sizes of 36 or more in the huntingtin gene (HTT) cause Huntington disease. Apart from neuropsychiatric complications, the disease is also accompanied by metabolic dysregulation and weight loss, which contribute to a progressive functional decline. Recent studies also reported an association between repeats below the pathogenic threshold (<36) for Huntington’s disease and body mass index (BMI), suggesting that HTT repeat sizes in the non-pathogenic range are associated with metabolic dysregulation. 
															In this study we hypothesized that HTT repeat sizes <36 are associated with metabolite levels, possibly mediated through reduced BMI. We pooled data from three European cohorts (n=10,228) with genotyped HTT CAG repeat size and metabolomic measurements. All 145 metabolites were measured on the same targeted platform in all studies. Multilevel mixed-effects analysis using the CAG repeat size in HTT identified 67 repeat size-metabolite associations. Overall, the metabolomic profile associated with larger CAG repeat sizes in HTT were unfavorable—similar to those of higher risk of coronary artery disease and type 2 diabetes—and included elevated levels of amino acids, fatty acids, LDL, VLDL and IDL related metabolites whilst with decreased levels of very large HDL related metabolites. Furthermore, the associations of 50 metabolites, in particular specific very large HDL related metabolites, were mediated by lower BMI. However, no mediation effect was found for 17 metabolites related to LDL and IDL. In conclusion, our findings indicate that large non-pathogenic CAG repeat sizes in HTT are associated with an unfavorable metabolomic profile despite their association with a lower BMI.
															<br><br>
															<b>Keywords:</b> Huntingtin gene, CAG repeats, Lipoproteins, Metabolomics, Multilevel mixed-effects analysis </p>
													</div>
												</div>
											</div>
											<div class="modal-footer">
												<a href="https://academic.oup.com/hmg/advance-article/doi/10.1093/hmg/ddad020/7009474" target="_blank">
													<button type="button" class="btn btn-sm btn-default" >Full Text</button> </a>

												<a href="https://pubmed.ncbi.nlm.nih.gov/36715614/" target="_blank">
													<button type="button" class="btn btn-sm btn-default" >PubMed</button> </a>

												<button type="button" class="btn btn-sm btn-default" data-dismiss="modal">Close</button>
											</div>
										</div>
									</div>
								</div>
								<!-- Modal end -->
							</div>
						
					


							
							<div class="col-sm-6 col-md-3 isotope-item Proteomics Methodology">
								<div class="image-box">
									<div class="overlay-container">
										<img src="images/portfolio-2.png" alt="">
										<a class="overlay" data-toggle="modal" data-target="#project-2">
											<i class="fa fa-search-plus"></i>
											 
										</a>
									</div>
									<a class="btn btn-default btn-block" data-toggle="modal" data-target="#project-2">Published</a>
								</div>
								<h6 class="text-center">Agreement of aptamer proteomics with standard methods for measuring venous thrombosis biomarkers</h6>

								<!-- Modal -->
								<div class="modal fade" id="project-2" tabindex="-1" role="dialog" aria-labelledby="project-2-label" aria-hidden="true">
									<div class="modal-dialog modal-lg">
										<div class="modal-content">
											<div class="modal-header">
												<button type="button" class="close" data-dismiss="modal"><span aria-hidden="true">&times;</span><span class="sr-only">Close</span></button>
												<h4 class="modal-title" id="project-2-label">Agreement of aptamer proteomics with standard methods for measuring venous thrombosis biomarkers (2021)
												</h4>
											</div>
											<div class="modal-body">
												
												<div class="row">												 
													<div class="col-md-12">
														<h7><i>Faquih T, Mook-Kanamori DO, Rosendaal FR, Baglin T, Willems van Dijk K, van Hylckama Vlieg A. Agreement of aptamer proteomics with standard methods for measuring venous thrombosis biomarkers. Res Pract Thromb Haemost. 2021 May 4;5(4):e12526. doi: 10.1002/rth2.12526. PMID: 34013156; PMCID: PMC8110437.</i>
														</h7><br>
														<p><b>Abstract</b><br>
															Background: Venous thromboembolism (VTE) is a complex disease with an incidence rate of about 1 in 1000 per year. Despite the availability of validated biomarkers for VTE, unprovoked events account for 50% of first events. Therefore, emerging high-throughput proteomics are promising methods for the expansion of VTE biomarkers. One such promising high-throughput platform is SomaScan, which uses a large library of synthetic oligonucleotide ligands known as aptamers to measure thousands of proteins.
	
															Objective: The aim of this study was to evaluate the viability of the aptamer-based SomaScan platform for VTE studies by examining its agreement with standard laboratory methods.
	
															Methods: We examined the agreement between eight established VTE biomarkers measured by SomaScan and standard laboratory immunoassay and viscosity-based instruments in 54 individuals (27 cases and 27 controls) from the Thrombophilia, Hypercoagulability and Environmental Risks in Venous Thromboembolism study. We performed the agreement analysis by using a regression model and predicting the estimates and the 95% prediction interval (PI) of the laboratory instrument values using SomaScan values.
	
															Results: SomaScan measurements exhibited overall poor agreement, particularly for D-dimer (average fit, 492.7 ng/mL; 95% PI, 110.0-1998.2) and fibrinogen (average fit, 3.3 g/L; 95% PI, 2.0-4.7).
	
															Conclusion: Our results indicate that SomaScan measurement had poor agreement with the standard laboratory measurements. These results may explain why some genome-wide association studies with VTE proteins measured by SomaScan did not confirm previously identified loci. Therefore, SomaScan should be considered with caution in VTE studies.
															<br><br>
															<b>Keywords:</b> aptamers; biomarkers; blood coagulation factors; immunoassay; nucleotide; proteomics; venous thrombosis.</p>
													</div>
												</div>
											</div>
											<div class="modal-footer">
												<a href="https://doi.org/10.1002/rth2.12526" target="_blank">
													<button type="button" class="btn btn-sm btn-default" >Full Text</button> </a>

												<a href="https://pubmed.ncbi.nlm.nih.gov/34013156/" target="_blank">
												<button type="button" class="btn btn-sm btn-default" >PubMed</button> </a>

												<button type="button" class="btn btn-sm btn-default" data-dismiss="modal">Close</button>
											</div>
										</div>
									</div>
								</div>
								<!-- Modal end -->
							</div>
							


							<!--hand-->
														
							<div class="col-sm-6 col-md-3 isotope-item Prediction Metabolomics" style="top: 300px">
								<div class="image-box">
									<div class="overlay-container">
										<img src="images/osteoarthritis.jpg" alt="">
										<a class="overlay" data-toggle="modal" data-target="#project-11">
											<i class="fa fa-search-plus"></i> 
										</a>
									</div>
									<a class="btn btn-default btn-block" data-toggle="modal" data-target="#project-11">Published</a>
								</div>
								<h6 class="text-center">The lipid profile for the prediction of prednisolone treatment response in patients with inflammatory hand osteoarthritis: The HOPE study
								<br><br></h6>

								<!-- Modal -->
								<div class="modal fade" id="project-11" tabindex="-1" role="dialog" aria-labelledby="project-11-label" aria-hidden="true">
									<div class="modal-dialog modal-lg">
										<div class="modal-content">
											<div class="modal-header">
												<button type="button" class="close" data-dismiss="modal"><span aria-hidden="true">&times;</span><span class="sr-only">Close</span></button>
												<h4 class="modal-title" id="project-11-label">The lipid profile for the prediction of prednisolone treatment response in patients with inflammatory hand osteoarthritis: The HOPE study</h4>
											</div>
											<div class="modal-body">
												
												<div class="row">												 
													<div class="col-md-12">
														<h7><i>Loef M, Faquih TO, von Hegedus JH, Ghorasaini M, Ioan-Facsinay A, Kroon FPB, Giera M, Kloppenburg M. The lipid profile for the prediction of prednisolone treatment response in patients with inflammatory hand osteoarthritis: The HOPE study. Osteoarthr Cartil Open. 2021 Apr 22;3(4):100167. doi: 10.1016/j.ocarto.2021.100167. PMID: 36474761; PMCID: PMC9718086.</i>
														</h7><br>
														<p><b>Abstract</b><br>
															Objective: To explore the use of lipidomics for prediction of prednisolone treatment response in patients with inflammatory hand osteoarthritis.

			Design: The Hand Osteoarthritis Prednisolone Efficacy (HOPE) study included patients (n ​= ​92) with symptomatic inflammatory hand osteoarthritis, fulfilling the ACR criteria. The present analyses comprised only patients randomized to prednisolone treatment (10 ​mg daily, n ​= ​40). Response to prednisolone treatment was defined according to the OARSI-OMERACT responder criteria at six weeks. Baseline blood samples were obtained non-fasted. Lipid species were quantified in erythrocytes with the Lipidyzer™ platform (Sciex). Oxylipins were analyzed in plasma using an in-house LC-MS/MS platform. Elastic net regularized regression was used to predict prednisolone treatment response based on common patient characteristics alone and including the patients' lipid profile. ROC analyses with 1000 bootstrapped area under the curve (AUC) was used to determine the discriminatory accuracy of the models.

			Results: Among included patients, 78% fulfilled the OARSI-OMERACT responder criteria. From the general patient characteristics, elastic net selected baseline hand function as only predictor of treatment response, with an AUC of 0.78 (0.56; 0.97). Addition of lipidomics resulted in an AUC of 0.92 (0.78; 0.99) and 0.85 (0.65; 0.98) for inclusion of the Lipidyzer™ platform and oxylipin platform, respectively.

			Conclusion: Our results suggest that the patients' lipid profile may improve the discriminative accuracy of the prediction of prednisolone treatment response in patients with inflammatory hand osteoarthritis compared to prediction by commonly measured patient characteristics alone. Hence, lipidomics may be a promising field for biomarker discovery for prediction of anti-inflammatory treatment response.
															<br><br>
															<b>Keywords:</b>
															Hand osteoarthritis; Lipids; Metabolomics; Prediction; Prednisolone; Treatment response</p>
													</div>
												</div>
											</div>
											<div class="modal-footer">
												<a href="https://pubmed.ncbi.nlm.nih.gov/36474761/" target="_blank">
													<button type="button" class="btn btn-sm btn-default" >pubmed</button> </a>

												<button type="button" class="btn btn-sm btn-default" data-dismiss="modal">Close</button>
											</div>
										</div>
									</div>
								</div>
								<!-- Modal end -->
							</div>

 							<div class="col-sm-6 col-md-3 isotope-item Genomics" >
								<div class="image-box">
									<div class="overlay-container">
										<img src="images/portfolio-4.png" alt="">
										<a class="overlay" data-toggle="modal" data-target="#project-4">
											<i class="fa fa-search-plus"></i> 
										</a>
									</div>
									<a class="btn btn-default btn-block" data-toggle="modal" data-target="#project-4">Published</a>
								</div>
								<h6 class="text-center">Saudi Human Genome Project</h6>

								<!-- Modal -->
								<div class="modal fade" id="project-4" tabindex="-1" role="dialog" aria-labelledby="project-4-label" aria-hidden="true">
									<div class="modal-dialog modal-lg">
										<div class="modal-content">
											<div class="modal-header">
												<button type="button" class="close" data-dismiss="modal"><span aria-hidden="true">&times;</span><span class="sr-only">Close</span></button>
												<h4 class="modal-title" id="project-4-label">Saudi Human Genome Project (2013-2019)</h4>
											</div>
											<div class="modal-body">
												
												<div class="row">												 
													<div class="col-md-12">
														<h3>Project Description</h3>
														<p>During my time in at King Faisal Specialist Hospital and Research Center (Saudi Arabia) I Worked as bioinformatician as part of the Saudi Human Genome Project. The goal of the project was to "To determine the variants underlying genetic disease in the Kingdom of Saudi Arabia, localize and develop genetic sequencing technologies, the field of genomics and bioinformatics in order to lay the foundation of personalized medicine". <br>
															My work involved using and developing tools for exome and full genome analysis and annotation and database development and management. 
															
															<br/><br/>

															<b>Notable Publications:</b>

															<br><br><h7><i>Monies D, et al. The landscape of genetic diseases in Saudi Arabia based on the first 1000 diagnostic panels and exomes. Hum Genet. 2017 Aug;136(8):921-939. doi: 10.1007/s00439-017-1821-8. Epub 2017 Jun 9. PMID: 28600779; PMCID: PMC5502059.</i></h7> 

															<br><br><h7><i>Yemni EA, et al. Integrated Analysis of Whole Exome Sequencing and Copy Number Evaluation in Parkinson's Disease. Sci Rep. 2019 Mar 4;9(1):3344. doi: 10.1038/s41598-019-40102-x. PMID: 30833663; PMCID: PMC6399448.</i></h7> 

															<br><br><h7><i>Abouelhoda M, Faquih T, El-Kalioby M, Alkuraya FS. Revisiting the morbid genome of Mendelian disorders. Genome Biol. 2016 Nov 24;17(1):235. doi: 10.1186/s13059-016-1102-1. PMID: 27884173; PMCID: PMC5123336.</i></h7> 
															</p>
													</div>
												</div>
											</div>
											<div class="modal-footer">
												<button type="button" class="btn btn-sm btn-default" data-dismiss="modal">Close</button>
											</div>
										</div>
									</div>
								</div>
								<!-- Modal end -->
							</div> 
							


						</div>
						<!-- portfolio items end -->
					
					</div>
				</div>
			</div>
		</section>
		<!-- section end -->

		<div class="default-bg colord secPadding">
			<div class="container">

				</div>
			</div>
		</div>


		<!-- footer start --> 
		<footer id="footer">
<!-- .footer start --> 

<!-- .footer end -->
			<!-- .subfooter start --> 
			<div class="subfooter">
				<div class="container">
					<div class="row">
						
						<div class="col-md-12">
							
							<p class="text-center">Copyright © 2025 Tariq Faquih.</p>
						</div>
					</div>
				</div>
			</div>
			<!-- .subfooter end -->

		</footer>
		<!-- footer end -->

		<!-- JavaScript --> 
		<script type="text/javascript" src="plugins/jquery.min.js"></script>
		<script type="text/javascript" src="bootstrap/js/bootstrap.min.js"></script>
		<script type="text/javascript" src="plugins/modernizr.js"></script>
		<script type="text/javascript" src="plugins/isotope/isotope.pkgd.min.js"></script>
		<script type="text/javascript" src="plugins/jquery.backstretch.min.js"></script>
		<script type="text/javascript" src="plugins/jquery.appear.js"></script>

		<!-- Custom Scripts -->
		<script type="text/javascript" src="js/custom.js"></script>

	</body>
</html>
