<!DOCTYPE html>
<html lang="en">

<head>

    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <meta name="description" content="Hepatic triglyceride content Atlas">
    <meta name="author" content="Tariq Faquih">

    <title>Hepatic triglyceride content is intricately associated with numerous metabolites and biochemical pathways</title>

    <!-- Bootstrap Core CSS -->
    <link href="./css/bootstrap.min.css" rel="stylesheet" type="text/css">

    <!-- Custom CSS -->
    <link href="./css/stylish-portfolio.css" rel="stylesheet" type="text/css">
    <!-- Custom Fonts -->
    
    <link href="./font-awesome/css/font-awesome.min.css" rel="stylesheet" type="text/css">
    <link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro:300,400,700,300italic,400italic,700italic" rel="stylesheet">






    <!-- HTML5 Shim and Respond.js IE8 support of HTML5 elements and media queries -->
    <!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
    <!--[if lt IE 9]>
        <script src="https://oss.maxcdn.com/libs/html5shiv/3.7.0/html5shiv.js"></script>
        <script src="https://oss.maxcdn.com/libs/respond.js/1.4.2/respond.min.js"></script>
    <![endif]-->

</head>

<body>

    <!-- Navigation -->
    <a id="menu-toggle" href="#" class="btn btn-dark btn-lg toggle"><i class="fa fa-bars"></i></a>
    <nav id="sidebar-wrapper">
        <ul class="sidebar-nav">
            <a id="menu-close" href="#" class="btn btn-light btn-lg pull-right toggle"><i class="fa fa-times"></i></a>
            <li class="sidebar-brand">
                <a href="#top">Menu</a>
            </li>
            <li>
                <a href="#top">Home</a>
            </li>
            <li>
                <a href="#Abstract">Abstract</a>
            </li>
            <li>
                <a href="#Methods">Methods</a>
            </li>
            <li>
                <a href="#Networks">Networks</a>
            </li>

        </ul>
    </nav>

    <!-- Header -->
    <header id="top" class="header">
        <div class="text-vertical-center">
            <h1>Hepatic triglyceride content is intricately associated with numerous metabolites and biochemical pathways</h1>


            <br>

        </div>
    </header>

    <!-- About -->
    <section id="Abstract" class="Abstract">
        <div class="container">
            <div class="row">
                <h2 class="text-center">Abstract</h2>
                <hr class="small">

                <div class="col-lg-12 text-justify">
                    <p class="lead"><b>Background</b>
                        Non-alcoholic fatty liver disease (NAFLD) is characterized by the pathological accumulation of triglycerides in hepatocytes and is associated with insulin resistance, atherogenic dyslipidaemia and cardiometabolic diseases. Thus far, the extent of metabolic dysregulation associated with hepatic triglyceride accumulation has not been fully addressed. In this study, we aimed to identify metabolites associated with hepatic triglyceride content (HTGC) and map these associations using network analysis. <br>
                        <b>Methods</b>
                        To gain insight in the spectrum of metabolites associated with hepatic triglyceride accumulation, we performed a comprehensive plasma metabolomics screening of 1363 metabolites in apparently healthy middle aged (age 45–65) individuals (N = 496) in whom HTGC was measured by proton magnetic resonance spectroscopy. An atlas of metabolite–HTGC associations, based on univariate results, was created using correlation-based Gaussian graphical model (GGM) and genome scale metabolic model network analyses. Pathways associated with the clinical prognosis marker fibrosis 4 (FIB-4) index were tested using a closed global test. <br>

                        <b>Results</b>
                        Our analyses revealed that 118 metabolites were univariately associated with HTGC (p-value <6.59 × 10−5), including 106 endogenous, 1 xenobiotic and 11 partially characterized/uncharacterized metabolites. These associations were mapped to several biological pathways including branched amino acids (BCAA), diglycerols, sphingomyelin, glucosyl-ceramide and lactosyl-ceramide. We also identified a novel possible HTGC-related pathway connecting glutamate, metabolonic lactone sulphate and X-15245 using the GGM network. These pathways were confirmed to be associated with the FIB-4 index as well. <br>

                        <b>Conclusion</b>
                        The combined network and pathway analyses indicated extensive associations between BCAA and the lipids pathways with HTGC and the FIB-4 index. Moreover, we report a novel pathway glutamate-metabolonic lactone sulphate-X-15245 with a potential strong association with HTGC. These findings can aid elucidating HTGC metabolomic profiles and provide insight into novel drug targets for fibrosis-related outcomes.
                        </a></p>                        
                        

                        
                </div>
                <div class="col-lg-12 text-center">
                    <hr class="small">
                    <h3>Our paper is published in the journal Liver International</h3>
                    <a href="https://pubmed.ncbi.nlm.nih.gov/37017544/" class="btn btn-lg btn-dark">PubMed Page</a><br><br>
                </div>
                

            </div>

            <!-- /.row -->
        </div>
        
        <!-- /.container -->
    </section>


    <!-- Methods -->
    <!-- The circle icons use Font Awesome's stacked icon classes. For more information, visit http://fontawesome.io/examples/ -->
    <section id="Methods" class="services bg-primary">
        <div class="container">
            <div class="row text-center">
                <div class="col-lg-10 col-lg-offset-1">
                    <h2>Methods</h2>
                    <hr class="small">
                    <div class="row">
                        <div class="col-md-3 col-sm-6">

                        </div>
                        <div class="col-md-3 col-sm-6" >
                            <div class="service-item">
                                <span class="fa-stack fa-4x">
                                    <img class="" src="img/noun_matrix_2640648.png" width="100"></i>
                            </span>
                                <h4>
                                    <strong>Gaussian Graphical Model (GGM)</strong>
                                </h4>
                                <p class="text-justify">Significant metabolites from the main analysis and the sex stratified analysis were used to develop the GGM networks separately. To create the network, we used the full measurements of the metabolites from our imputed dataset. Since we used multiple imputation, we could only select one imputed dataset. The correlations between the metabolites and HTGC were calculated using sparse Gaussian Graphical Models with graphical LASSO using the glasso function in R (qgraph R package version 1.6.9) coupled with the Extended Bayesian Information Criterium (EBIC) to tune the LASSO shrinkage factor. EBIC has been reported in simulation studies to perform better than the standard BIC, particularly for controlling false positivity rates. Here, we set the EBIC tuning parameter to the default value of 0.5 (wherein a value of 0 uses standard BIC) for a balanced tuning . Based on the correlation coefficients, edges were drawn between the correlated metabolite nodes with the width corresponding to the coefficient strength. The size of the nodes was determined by the absolute value of the effect estimate from the multivariable linear regression analysis. The metabolite nodes were colored based on their biochemical super and sub pathway classes and organized in ascending order based on their p-values. Visualization and layout of the networks were created in Gephi and exported as an interactive HTML5 using the sigmaExporter plugin.  </p>
                                <a href="https://onlinelibrary.wiley.com/doi/10.1111/liv.15575#support-information-section" class="btn btn-light">Learn More</a>
                            </div>
                        </div>
                        <div class="col-md-3 col-sm-6">
                            <div class="service-item">
                                <span class="fa-stack fa-4x">
                                <img class="" src="img/noun_network and science_3173333.png" width="100"></i>
                            </span>
                                <h4>
                                    <strong>Genome Scale Metabolic Model (GSMM)</strong>
                                </h4>
                                <p class="text-justify">We mapped the metabolite names onto a curated version of the Human1 genome-scale metabolic model (17) and calculated the biochemical interconversions between all pairs of metabolites that were measured on the Metabolon platform on a liver-specific subset of the model. To facilitate the mapping of metabolomics data to the GSMM, we enriched the model with compound synonyms and external identifiers from the Chemical Entities of Biological Interest (ChEBI) database (18), where ChEBI identifiers and synonyms of conjugate acids and bases were also included. Of the 1067 metabolites with known identity that were quantified, 436 mapped properly to the GSMM. All reactions and compounds in Human1 were checked for mass and redox balance and were adjusted when necessary. 
                                    Biochemical interactions between metabolites were determined by converting the GSMM into a weighted directed graph where nodes represent metabolites and edges represent reactions. Subsequently all reaction paths between the measured metabolites that involved one or two reaction steps were determined using a generic path finding algorithm that was developed in house. 
                                    
                                    This resulted in a network of connected metabolites that we integrated with biochemical reaction and pathway knowledge into an interactive HTML/JavaScript document. The results from the network analysis include measured associated metabolites as well as relevant intermediate metabolites regardless of whether they were measured in the dataset. In addition, the network shows the directionality of the biological reaction paths for the biosynthesis and degradation of the metabolites and includes details regarding the involved genes and intermediate reaction steps. Furthermore, for all metabolites and genes, also those not included in the network, hyperlinks to external databases and gene expression profiles are provided.  Reaction information was enriched by importing tissue-specific gene expression from the Human Protein Atlas (19) and Genotype-Tissue Expression (GTEx) project. 
                                    All computations described in this section were performed in MATLAB R2019b.
                                    </p>
                                <a href="https://onlinelibrary.wiley.com/doi/10.1111/liv.15575#support-information-section" class="btn btn-light">Learn More</a>
                            </div>
                        </div>
                        <div class="col-md-3 col-sm-6">

                        </div>
                    </div>
                    <!-- /.row (nested) -->
                </div>
                <!-- /.col-lg-10 -->
            </div>
            <!-- /.row -->
        </div>
        <!-- /.container -->
    </section>



    <!-- Networks -->
    <section id="Networks" class="Networks">
        <div class="container">
            <div class="row">
                <div class="col-lg-10 col-lg-offset-1 text-center">
                    <h2>Networks</h2>
                    <hr class="small">
                    <div class="row">
                        <div class="col-md-6">
                            <div class="portfolio-item">
                                <a href="https://tofaquih.github.io/AtlasLiver/Networks/LiverFatNetworks/Model2/SexAdj/network">
                                    <img class="img-portfolio img-responsive" src="img/GMM_main.png">
                                </a>
                            </div>
                        </div>
                        <div class="col-md-6">
                            <div class="portfolio-item">
                                <a href="https://tofaquih.github.io/AtlasLiver/Networks/LiverFatNetworks/Model2/Female/network">
                                    <img class="img-portfolio img-responsive" src="img/GMM_Women.png">
                                </a>
                            </div>
                        </div>
                        <div class="col-md-6">
                            <div class="portfolio-item">
                                <a href="https://tofaquih.github.io/AtlasLiver/Networks/LiverFatNetworks/Model2/Male/network">
                                    <img class="img-portfolio img-responsive" src="img/GMM_men.png">
                                </a>
                            </div>
                        </div>
                        <div class="col-md-6">
                            <div class="portfolio-item">
                                <a href="https://tofaquih.github.io/AtlasLiver/Networks/LiverFatNetworks/HRN_pathway_browser_Faquih_NAFLD_biomarkers3/HRN_pathway_browser.html">
                                    <img class="img-portfolio img-responsive" src="img/gsmm.png">
                                </a>
                            </div>
                        </div>
                        
                    </div>

                    
                </div>
                <!-- /.col-lg-10 -->
            </div>
            
            <!-- /.row -->
        </div>
        <!-- /.container -->
    </section>

    <!-- Call to Action -->
    <aside class="call-to-action bg-primary">
        <div class="container">
            <div class="row">
                <div class="col-lg-12 text-center">
                    <h3>For more information regarding the paper and our other projects:</h3>
                    <a href="https://onlinelibrary.wiley.com/doi/10.1111/liv.15575" class="btn btn-lg btn-light">Full Text</a>
                    <a href="https://pubmed.ncbi.nlm.nih.gov/37017544/" class="btn btn-lg btn-dark">Pubmed</a>
                    <a href="https://tofaquih.github.io/" class="btn btn-lg btn-dark">My Webpage</a>
                </div>
            </div>
        </div>
    </aside>

  

    <!-- Footer -->
    <footer>
        <div class="container">
            <div class="row">
                <div class="col-lg-10 col-lg-offset-1 text-center">
                    <h4><strong>Tariq Faquih</strong>
                    </h4>
                    <p>Leiden<br>Netherlands, 2333 ZA</p>
                    <ul class="list-unstyled">

                    </ul>

                    <p class="text-muted">Copyright &copy; tofaquih.github.io/AtlasLiver/ 2023<br>Leiden University Medical Center </p>
                </div>
            </div>
        </div>
    </footer>

    <!-- jQuery -->
    <script src="js/jquery.js"></script>

    <!-- Bootstrap Core JavaScript -->
    <script src="js/bootstrap.min.js"></script>

    <!-- Custom Theme JavaScript -->
    <script>
    // Closes the sidebar menu
    $("#menu-close").click(function(e) {
        e.preventDefault();
        $("#sidebar-wrapper").toggleClass("active");
    });

    // Opens the sidebar menu
    $("#menu-toggle").click(function(e) {
        e.preventDefault();
        $("#sidebar-wrapper").toggleClass("active");
    });

    // Scrolls to the selected menu item on the page
    $(function() {
        $('a[href*=#]:not([href=#])').click(function() {
            if (location.pathname.replace(/^\//, '') == this.pathname.replace(/^\//, '') || location.hostname == this.hostname) {

                var target = $(this.hash);
                target = target.length ? target : $('[name=' + this.hash.slice(1) + ']');
                if (target.length) {
                    $('html,body').animate({
                        scrollTop: target.offset().top
                    }, 1000);
                    return false;
                }
            }
        });
    });
    </script>

</body>

</html>

